2024-06-04 13:47:34,332 - INFO - Starting to fetch search results for query: This work was supported in part through the NYU IT High Performance Computing resources, services, and staff expertise from page 0
2024-06-04 13:47:34,760 - WARNING - Row does not contain enough cells
2024-06-04 13:47:35,180 - WARNING - Row does not contain enough cells
2024-06-04 13:47:35,692 - WARNING - Row does not contain enough cells
2024-06-04 13:47:36,768 - ERROR - Connection refused for proxy 203.205.9.105:8080
2024-06-04 13:47:37,014 - ERROR - Connection refused for proxy 35.185.196.38:3128
2024-06-04 13:47:37,676 - ERROR - Connection refused for proxy 172.96.117.205:38001
2024-06-04 13:47:37,901 - ERROR - Connection refused for proxy 117.250.3.58:8080
2024-06-04 13:47:38,351 - ERROR - Connection refused for proxy 72.10.160.91:17137
2024-06-04 13:47:38,957 - ERROR - Connection refused for proxy 67.43.228.253:1503
2024-06-04 13:47:39,433 - ERROR - Connection refused for proxy 8.219.97.248:80
2024-06-04 13:47:40,524 - ERROR - Connection refused for proxy 72.10.160.172:16679
2024-06-04 13:47:40,752 - ERROR - Connection refused for proxy 185.217.136.67:1337
2024-06-04 13:47:41,176 - ERROR - Connection refused for proxy 51.145.176.250:8080
2024-06-04 13:47:41,426 - ERROR - General error for proxy 20.235.47.207:3128: HTTPConnectionPool(host='20.235.47.207', port=3128): Read timed out. (read timeout=5)
2024-06-04 13:47:41,590 - ERROR - Connection refused for proxy 67.43.228.251:2081
2024-06-04 13:47:41,719 - ERROR - Connection refused for proxy 171.251.92.87:5404
2024-06-04 13:47:41,908 - ERROR - Connection refused for proxy 36.111.143.8:15777
2024-06-04 13:47:42,028 - ERROR - Connection refused for proxy 61.97.191.101:9070
2024-06-04 13:47:42,258 - ERROR - General error for proxy 20.37.207.8:8080: HTTPConnectionPool(host='20.37.207.8', port=8080): Read timed out. (read timeout=5)
2024-06-04 13:47:42,631 - ERROR - Connection refused for proxy 34.92.250.88:11111
2024-06-04 13:47:42,692 - ERROR - Connection refused for proxy 130.245.128.193:8080
2024-06-04 13:47:42,908 - ERROR - Connection refused for proxy 41.65.251.47:1981
2024-06-04 13:47:44,206 - ERROR - Connection refused for proxy 67.43.236.18:15267
2024-06-04 13:47:44,440 - ERROR - Connection refused for proxy 72.10.160.170:33147
2024-06-04 13:47:44,810 - ERROR - Connection refused for proxy 67.43.228.252:3187
2024-06-04 13:47:45,077 - ERROR - Connection refused for proxy 67.43.227.227:17629
2024-06-04 13:47:45,527 - ERROR - Connection refused for proxy 72.10.160.90:20931
2024-06-04 13:47:45,763 - ERROR - Connection refused for proxy 66.188.181.143:8080
2024-06-04 13:47:46,183 - ERROR - Connection refused for proxy 203.189.88.156:80
2024-06-04 13:47:46,734 - ERROR - Connection refused for proxy 54.67.45.218:3128
2024-06-04 13:47:46,922 - ERROR - Connection refused for proxy 34.154.161.152:80
2024-06-04 13:47:47,185 - ERROR - Connection refused for proxy 20.44.190.150:3129
2024-06-04 13:47:47,541 - ERROR - Connection refused for proxy 20.204.212.76:3129
2024-06-04 13:47:47,918 - ERROR - Connection refused for proxy 20.204.214.79:3129
2024-06-04 13:47:48,233 - ERROR - General error for proxy 45.77.147.46:3128: HTTPConnectionPool(host='45.77.147.46', port=3128): Read timed out. (read timeout=5)
2024-06-04 13:47:48,625 - ERROR - Connection refused for proxy 20.204.212.45:3129
2024-06-04 13:47:48,778 - ERROR - Connection refused for proxy 20.44.188.17:3129
2024-06-04 13:47:49,255 - ERROR - Connection refused for proxy 20.219.176.57:3129
2024-06-04 13:47:50,079 - ERROR - Connection refused for proxy 50.28.7.107:80
2024-06-04 13:47:51,926 - ERROR - Connection refused for proxy 80.87.178.175:8080
2024-06-04 13:47:51,981 - ERROR - Connection refused for proxy 188.247.194.210:3128
2024-06-04 13:47:52,191 - ERROR - Connection refused for proxy 161.34.39.153:3128
2024-06-04 13:47:52,797 - ERROR - Connection refused for proxy 62.33.53.248:3128
2024-06-04 13:47:52,895 - ERROR - Connection refused for proxy 199.167.236.12:3128
2024-06-04 13:47:52,924 - ERROR - Connection refused for proxy 103.139.127.243:8080
2024-06-04 13:47:53,095 - ERROR - Connection refused for proxy 154.91.1.213:3129
2024-06-04 13:47:53,193 - ERROR - Connection refused for proxy 147.75.88.36:10001
2024-06-04 13:47:53,356 - ERROR - Connection refused for proxy 221.168.33.155:8080
2024-06-04 13:47:55,081 - ERROR - Connection refused for proxy 188.132.222.37:8080
2024-06-04 13:47:55,277 - ERROR - Connection refused for proxy 47.243.92.199:3128
2024-06-04 13:47:55,891 - ERROR - General error for proxy 47.91.65.23:3128: HTTPConnectionPool(host='47.91.65.23', port=3128): Read timed out. (read timeout=5)
2024-06-04 13:47:56,218 - ERROR - Connection refused for proxy 122.155.165.191:3128
2024-06-04 13:47:56,608 - ERROR - Connection refused for proxy 152.70.235.185:9002
2024-06-04 13:47:56,637 - ERROR - Connection refused for proxy 111.206.0.99:8181
2024-06-04 13:47:56,921 - ERROR - Connection refused for proxy 182.253.109.59:8080
2024-06-04 13:47:57,199 - ERROR - Connection refused for proxy 189.240.60.164:9090
2024-06-04 13:47:58,107 - ERROR - Connection refused for proxy 109.164.38.189:2306
2024-06-04 13:47:58,666 - ERROR - Connection refused for proxy 178.48.68.61:18080
2024-06-04 13:47:59,312 - ERROR - Connection refused for proxy 186.96.50.113:999
2024-06-04 13:47:59,333 - ERROR - General error for proxy 31.45.237.146:8080: HTTPConnectionPool(host='31.45.237.146', port=8080): Read timed out. (read timeout=5)
2024-06-04 13:48:00,095 - ERROR - Connection refused for proxy 103.247.122.166:8080
2024-06-04 13:48:00,257 - ERROR - Connection refused for proxy 72.10.160.94:3901
2024-06-04 13:48:00,265 - ERROR - Connection refused for proxy 20.204.214.23:3129
2024-06-04 13:48:01,225 - ERROR - Connection refused for proxy 191.243.46.166:43241
2024-06-04 13:48:01,641 - ERROR - Connection refused for proxy 149.28.128.233:10000
2024-06-04 13:48:01,934 - ERROR - Connection refused for proxy 38.65.172.10:999
2024-06-04 13:48:02,278 - ERROR - Connection refused for proxy 67.43.227.230:5097
2024-06-04 13:48:02,411 - ERROR - Connection refused for proxy 67.43.236.22:18891
2024-06-04 13:48:02,803 - ERROR - General error for proxy 185.191.236.162:3128: HTTPConnectionPool(host='185.191.236.162', port=3128): Read timed out. (read timeout=5)
2024-06-04 13:48:04,314 - ERROR - Connection refused for proxy 154.127.240.120:64002
2024-06-04 13:48:05,484 - ERROR - General error for proxy 146.56.154.83:21000: HTTPConnectionPool(host='146.56.154.83', port=21000): Read timed out. (read timeout=5)
2024-06-04 13:48:06,658 - ERROR - Connection refused for proxy 154.127.240.119:64006
2024-06-04 13:48:06,951 - ERROR - Connection refused for proxy 154.127.240.124:64001
2024-06-04 13:48:07,202 - ERROR - Connection refused for proxy 189.240.60.163:9090
2024-06-04 13:48:07,292 - ERROR - Connection refused for proxy 154.127.240.114:64002
2024-06-04 13:48:07,419 - ERROR - Connection refused for proxy 154.127.240.115:64001
2024-06-04 13:48:07,620 - ERROR - General error for proxy 49.254.107.184:28766: HTTPConnectionPool(host='49.254.107.184', port=28766): Read timed out. (read timeout=5)
2024-06-04 13:48:07,665 - ERROR - Connection refused for proxy 154.21.201.49:8888
2024-06-04 13:48:07,805 - ERROR - Connection refused for proxy 154.127.240.117:64001
2024-06-04 13:48:08,872 - ERROR - Connection refused for proxy 189.240.60.171:9090
2024-06-04 13:48:10,488 - ERROR - Connection refused for proxy 114.129.2.82:8081
2024-06-04 13:48:11,675 - ERROR - Connection refused for proxy 20.27.86.185:8080
2024-06-04 13:48:11,752 - ERROR - Connection refused for proxy 87.247.186.40:1080
2024-06-04 13:48:11,953 - ERROR - Connection refused for proxy 148.72.165.7:10529
2024-06-04 13:48:12,216 - ERROR - Connection refused for proxy 199.168.175.179:80
2024-06-04 13:48:12,295 - ERROR - Connection refused for proxy 43.134.32.184:3128
2024-06-04 13:48:12,402 - ERROR - Connection refused for proxy 124.70.78.157:8000
2024-06-04 13:48:12,434 - ERROR - Connection refused for proxy 43.133.59.220:3128
2024-06-04 13:48:12,622 - ERROR - Connection refused for proxy 91.108.130.111:32650
2024-06-04 13:48:12,669 - ERROR - Connection refused for proxy 66.70.235.23:5454
2024-06-04 13:48:12,824 - ERROR - Connection refused for proxy 94.142.137.203:8080
2024-06-04 13:48:15,501 - ERROR - Connection refused for proxy 84.47.161.3:8118
2024-06-04 13:48:15,533 - ERROR - Connection refused for proxy 2024-06-04:312
2024-06-04 13:48:16,685 - ERROR - Connection refused for proxy 182.253.140.250:8080
2024-06-04 13:48:17,309 - ERROR - Connection refused for proxy 103.168.44.210:3127
2024-06-04 13:48:17,419 - ERROR - Connection refused for proxy 186.125.218.188:999
2024-06-04 13:48:17,451 - ERROR - Connection refused for proxy 181.119.84.229:999
2024-06-04 13:48:17,465 - ERROR - Connection refused for proxy 2024-06-03:352
2024-06-04 13:48:17,634 - ERROR - Connection refused for proxy 118.99.119.11:8080
2024-06-04 13:48:17,681 - ERROR - Connection refused for proxy 202.93.244.38:8080
2024-06-04 13:48:18,213 - ERROR - Connection refused for proxy 2024-06-02:476
2024-06-04 13:48:18,244 - ERROR - Connection refused for proxy 2024-06-01:402
2024-06-04 13:48:18,321 - ERROR - General error for proxy 47.254.90.125:8800: HTTPConnectionPool(host='47.254.90.125', port=8800): Read timed out. (read timeout=5)
2024-06-04 13:48:19,411 - ERROR - Connection refused for proxy 2024-05-31:367
2024-06-04 13:48:20,034 - ERROR - Connection refused for proxy 380:91
2024-06-04 13:48:20,159 - ERROR - Connection refused for proxy 175:42
2024-06-04 13:48:20,175 - ERROR - Connection refused for proxy 58:14
2024-06-04 13:48:21,790 - ERROR - Connection refused for proxy 216.137.184.253:80
2024-06-04 13:48:22,468 - ERROR - Connection refused for proxy 50.174.145.9:80
2024-06-04 13:48:22,640 - ERROR - Connection refused for proxy 50.172.75.125:80
2024-06-04 13:48:22,685 - ERROR - Connection refused for proxy 50.168.163.182:80
2024-06-04 13:48:23,217 - ERROR - Connection refused for proxy 50.232.104.86:80
2024-06-04 13:48:23,248 - ERROR - Connection refused for proxy 83.1.176.118:80
2024-06-04 13:48:23,326 - ERROR - Connection refused for proxy 82.119.96.254:80
2024-06-04 13:48:24,428 - ERROR - Connection refused for proxy 50.217.226.46:80
2024-06-04 13:48:25,189 - ERROR - Connection refused for proxy 50.145.24.176:80
2024-06-04 13:48:26,800 - ERROR - Connection refused for proxy 50.217.226.43:80
2024-06-04 13:48:27,479 - ERROR - Connection refused for proxy 50.223.239.175:80
2024-06-04 13:48:27,649 - ERROR - Connection refused for proxy 50.145.24.186:80
2024-06-04 13:48:27,696 - ERROR - Connection refused for proxy 50.217.226.42:80
2024-06-04 13:48:28,225 - ERROR - Connection refused for proxy 50.175.212.77:80
2024-06-04 13:48:28,255 - ERROR - Connection refused for proxy 50.172.75.121:80
2024-06-04 13:48:28,333 - ERROR - Connection refused for proxy 50.168.163.181:80
2024-06-04 13:48:28,820 - ERROR - Connection refused for proxy 172.96.117.205:38001
2024-06-04 13:48:29,340 - ERROR - General error for proxy 128.199.202.122:3128: Exceeded 30 redirects.
2024-06-04 13:48:29,438 - ERROR - Connection refused for proxy 50.168.72.112:80
2024-06-04 13:48:30,201 - ERROR - Connection refused for proxy 50.222.245.44:80
2024-06-04 13:48:32,487 - ERROR - Connection refused for proxy 50.168.72.114:80
2024-06-04 13:48:32,657 - ERROR - Connection refused for proxy 41.207.187.178:80
2024-06-04 13:48:32,703 - ERROR - Connection refused for proxy 50.174.7.157:80
2024-06-04 13:48:33,233 - ERROR - Connection refused for proxy 50.144.166.226:80
2024-06-04 13:48:33,265 - ERROR - Connection refused for proxy 190.58.248.86:80
2024-06-04 13:48:33,344 - ERROR - Connection refused for proxy 50.223.239.194:80
2024-06-04 13:48:33,829 - ERROR - Connection refused for proxy 50.175.212.74:80
2024-06-04 13:48:34,357 - ERROR - Connection refused for proxy 50.144.168.74:80
2024-06-04 13:48:34,451 - ERROR - Connection refused for proxy 50.217.226.41:80
2024-06-04 13:48:35,207 - ERROR - Connection refused for proxy 50.223.239.185:80
2024-06-04 13:48:37,504 - ERROR - Connection refused for proxy 50.172.75.122:80
2024-06-04 13:48:37,707 - ERROR - Connection refused for proxy 50.173.182.90:80
2024-06-04 13:48:37,826 - ERROR - General error for proxy 138.68.60.8:8080: Exceeded 30 redirects.
2024-06-04 13:48:38,200 - ERROR - Connection refused for proxy 50.222.245.50:80
2024-06-04 13:48:38,277 - ERROR - Connection refused for proxy 50.223.38.6:80
2024-06-04 13:48:38,354 - ERROR - Connection refused for proxy 50.168.72.117:80
2024-06-04 13:48:38,840 - ERROR - Connection refused for proxy 50.174.145.11:80
2024-06-04 13:48:38,988 - ERROR - General error for proxy 198.199.86.11:8080: Exceeded 30 redirects.
2024-06-04 13:48:39,362 - ERROR - Connection refused for proxy 50.222.245.42:80
2024-06-04 13:48:39,456 - ERROR - Connection refused for proxy 50.222.245.47:80
2024-06-04 13:48:40,221 - ERROR - Connection refused for proxy 50.221.230.186:80
2024-06-04 13:48:42,509 - ERROR - Connection refused for proxy 50.169.135.10:80
2024-06-04 13:48:42,710 - ERROR - Connection refused for proxy 50.172.75.126:80
2024-06-04 13:48:42,834 - ERROR - Connection refused for proxy 50.230.222.202:80
2024-06-04 13:48:43,286 - ERROR - Connection refused for proxy 50.174.7.156:80
2024-06-04 13:48:43,362 - ERROR - Connection refused for proxy 50.217.226.44:80
2024-06-04 13:48:43,680 - ERROR - Connection refused for proxy 188.132.209.245:80
2024-06-04 13:48:43,845 - ERROR - Connection refused for proxy 50.168.163.176:80
2024-06-04 13:48:44,001 - ERROR - Connection refused for proxy 50.168.72.116:80
2024-06-04 13:48:44,375 - ERROR - Connection refused for proxy 50.168.72.113:80
2024-06-04 13:48:44,471 - ERROR - Connection refused for proxy 50.174.7.153:80
2024-06-04 13:48:45,224 - ERROR - Connection refused for proxy 50.172.39.98:80
2024-06-04 13:48:47,515 - ERROR - Connection refused for proxy 195.23.57.78:80
2024-06-04 13:48:47,841 - ERROR - Connection refused for proxy 50.174.145.13:80
2024-06-04 13:48:48,292 - ERROR - Connection refused for proxy 50.222.245.43:80
2024-06-04 13:48:48,370 - ERROR - Connection refused for proxy 50.237.207.186:80
2024-06-04 13:48:48,682 - ERROR - Connection refused for proxy 50.207.199.81:80
2024-06-04 13:48:48,852 - ERROR - Connection refused for proxy 50.168.163.180:80
2024-06-04 13:48:49,005 - ERROR - Connection refused for proxy 50.223.246.237:80
2024-06-04 13:48:49,381 - ERROR - Connection refused for proxy 50.174.7.162:80
2024-06-04 13:48:49,488 - ERROR - Connection refused for proxy 50.222.245.40:80
2024-06-04 13:48:50,237 - ERROR - Connection refused for proxy 50.202.75.26:80
2024-06-04 13:48:51,034 - ERROR - Connection refused for proxy 8.219.97.248:80
2024-06-04 13:48:52,851 - ERROR - Connection refused for proxy 50.217.226.40:80
2024-06-04 13:48:52,853 - ERROR - Connection refused for proxy 0.0.0.0:80
2024-06-04 13:48:53,308 - ERROR - Connection refused for proxy 50.174.145.8:80
2024-06-04 13:48:53,386 - ERROR - Connection refused for proxy 50.168.72.115:80
2024-06-04 13:48:53,698 - ERROR - Connection refused for proxy 50.221.74.130:80
2024-06-04 13:48:53,855 - ERROR - Connection refused for proxy 50.231.172.74:80
2024-06-04 13:48:54,012 - ERROR - Connection refused for proxy 50.168.163.166:80
2024-06-04 13:48:54,385 - ERROR - Connection refused for proxy 50.171.187.51:80
2024-06-04 13:48:54,494 - ERROR - Connection refused for proxy 50.168.163.177:80
2024-06-04 13:48:55,241 - ERROR - Connection refused for proxy 50.239.72.19:80
2024-06-04 13:48:56,045 - ERROR - Connection refused for proxy 50.222.245.46:80
2024-06-04 13:48:57,862 - ERROR - Connection refused for proxy 50.172.75.123:80
2024-06-04 13:48:58,312 - ERROR - Connection refused for proxy 50.169.37.50:80
2024-06-04 13:48:58,391 - ERROR - Connection refused for proxy 50.174.7.152:80
2024-06-04 13:48:58,700 - ERROR - Connection refused for proxy 50.175.212.66:80
2024-06-04 13:48:58,871 - ERROR - Connection refused for proxy 50.174.7.154:80
2024-06-04 13:48:59,025 - ERROR - Connection refused for proxy 50.174.145.15:80
2024-06-04 13:48:59,394 - ERROR - Connection refused for proxy 32.223.6.94:80
2024-06-04 13:49:00,247 - ERROR - Connection refused for proxy 50.222.245.45:80
2024-06-04 13:49:01,061 - ERROR - Connection refused for proxy 50.174.7.158:80
2024-06-04 13:49:02,272 - ERROR - Connection refused for proxy 213.218.228.253:80
2024-06-04 13:49:02,878 - ERROR - Connection refused for proxy 50.239.72.17:80
2024-06-04 13:49:03,324 - ERROR - Connection refused for proxy 50.223.239.183:80
2024-06-04 13:49:03,402 - ERROR - Connection refused for proxy 50.174.7.159:80
2024-06-04 13:49:03,713 - ERROR - Connection refused for proxy 50.174.145.14:80
2024-06-04 13:49:03,883 - ERROR - Connection refused for proxy 50.175.212.76:80
2024-06-04 13:49:04,039 - ERROR - Connection refused for proxy 50.171.177.124:80
2024-06-04 13:49:04,238 - ERROR - Connection refused for proxy 35.185.196.38:3128
2024-06-04 13:49:04,410 - ERROR - Connection refused for proxy 50.223.239.191:80
2024-06-04 13:49:05,264 - ERROR - Connection refused for proxy 50.174.7.155:80
2024-06-04 13:49:06,070 - ERROR - Connection refused for proxy 50.145.24.181:80
2024-06-04 13:49:07,004 - ERROR - General error for proxy 43.152.112.135:12453: ("Connection broken: ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None)", ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None))
2024-06-04 13:49:07,276 - ERROR - Connection refused for proxy 50.239.72.18:80
2024-06-04 13:49:07,880 - ERROR - Connection refused for proxy 50.223.239.167:80
2024-06-04 13:49:08,410 - ERROR - Connection refused for proxy 68.185.57.66:80
2024-06-04 13:49:08,722 - ERROR - Connection refused for proxy 50.223.239.166:80
2024-06-04 13:49:08,894 - ERROR - Connection refused for proxy 156.239.50.176:3128
2024-06-04 13:49:09,532 - ERROR - Connection refused for proxy 156.239.55.151:3128
2024-06-04 13:49:12,008 - ERROR - Connection refused for proxy 156.239.55.49:3128
2024-06-04 13:49:12,594 - ERROR - Connection refused for proxy 104.165.169.150:3128
2024-06-04 13:49:14,542 - ERROR - Connection refused for proxy 156.239.48.96:3128
2024-06-04 13:49:14,649 - ERROR - Connection refused for proxy 104.165.127.165:3128
2024-06-04 13:49:15,068 - ERROR - Connection refused for proxy 104.164.183.176:3128
2024-06-04 13:49:15,984 - ERROR - Connection refused for proxy 156.239.55.41:3128
2024-06-04 13:49:16,312 - ERROR - Connection refused for proxy 104.165.169.240:3128
2024-06-04 13:49:16,715 - ERROR - Connection refused for proxy 104.252.131.217:3128
2024-06-04 13:49:17,022 - ERROR - Connection refused for proxy 104.165.127.40:3128
2024-06-04 13:49:19,904 - ERROR - General error for proxy 144.24.204.126:8118: ("Connection broken: ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None)", ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None))
2024-06-04 13:49:20,218 - ERROR - Connection refused for proxy 156.239.53.22:3128
2024-06-04 13:49:20,994 - ERROR - Connection refused for proxy 156.239.49.144:3128
2024-06-04 13:49:21,041 - ERROR - Connection refused for proxy 156.239.48.128:3128
2024-06-04 13:49:23,101 - ERROR - Connection refused for proxy 201.77.108.21:999
2024-06-04 13:49:23,316 - ERROR - General error for proxy 194.163.181.18:3128: HTTPConnectionPool(host='194.163.181.18', port=3128): Read timed out. (read timeout=5)
2024-06-04 13:49:23,501 - ERROR - Connection refused for proxy 104.164.183.246:3128
2024-06-04 13:49:26,012 - ERROR - Connection refused for proxy 104.165.169.180:3128
2024-06-04 13:49:27,168 - ERROR - Connection refused for proxy 104.164.183.71:3128
2024-06-04 13:49:28,118 - ERROR - Connection refused for proxy 104.252.131.170:3128
2024-06-04 13:49:28,330 - ERROR - Connection refused for proxy 156.239.48.10:3128
2024-06-04 13:49:28,424 - ERROR - Connection refused for proxy 104.164.183.129:3128
2024-06-04 13:49:31,828 - ERROR - Connection refused for proxy 156.239.49.138:3128
2024-06-04 13:49:33,130 - ERROR - Connection refused for proxy 156.239.48.180:3128
2024-06-04 13:49:34,344 - ERROR - Connection refused for proxy 156.239.52.44:3128
2024-06-04 13:49:35,362 - ERROR - Connection refused for proxy 104.252.131.10:3128
2024-06-04 13:49:35,903 - ERROR - Connection refused for proxy 104.165.127.101:3128
2024-06-04 13:49:37,109 - ERROR - Connection refused for proxy 51.210.19.141:80
2024-06-04 13:49:37,163 - ERROR - Connection refused for proxy 156.239.50.148:3128
2024-06-04 13:49:37,925 - ERROR - Connection refused for proxy 156.239.49.84:3128
2024-06-04 13:49:38,334 - ERROR - Connection refused for proxy 137.184.121.54:80
2024-06-04 13:49:38,917 - ERROR - Connection refused for proxy 104.164.183.215:3128
2024-06-04 13:49:40,069 - ERROR - Connection refused for proxy 67.43.227.228:12643
2024-06-04 13:49:40,371 - ERROR - Connection refused for proxy 4.155.2.13:9480
2024-06-04 13:49:40,859 - ERROR - Connection refused for proxy 210.212.39.138:8080
2024-06-04 13:49:40,863 - ERROR - Connection refused for proxy 201.218.200.115:8080
2024-06-04 13:49:41,347 - ERROR - Connection refused for proxy 116.203.28.43:80
2024-06-04 13:49:41,795 - ERROR - General error for proxy 85.209.154.148:33333: HTTPConnectionPool(host='85.209.154.148', port=33333): Read timed out. (read timeout=5)
2024-06-04 13:49:42,152 - ERROR - Connection refused for proxy 51.89.73.162:80
2024-06-04 13:49:42,166 - ERROR - Connection refused for proxy 117.54.114.96:80
2024-06-04 13:49:43,645 - ERROR - General error for proxy 183.100.14.134:8000: HTTPConnectionPool(host='183.100.14.134', port=8000): Read timed out. (read timeout=5)
2024-06-04 13:49:43,926 - ERROR - Connection refused for proxy 217.66.25.133:8060
2024-06-04 13:49:44,308 - ERROR - Connection refused for proxy 65.109.189.49:80
2024-06-04 13:49:44,390 - ERROR - General error for proxy 162.223.94.166:80: HTTPConnectionPool(host='162.223.94.166', port=80): Read timed out. (read timeout=5)
2024-06-04 13:49:44,577 - ERROR - Connection refused for proxy 185.159.153.109:80
2024-06-04 13:49:45,138 - ERROR - Connection refused for proxy 37.27.82.72:80
2024-06-04 13:49:45,545 - ERROR - Connection refused for proxy 37.27.29.181:80
2024-06-04 13:49:45,689 - ERROR - General error for proxy 161.35.70.249:3128: Exceeded 30 redirects.
2024-06-04 13:49:46,192 - ERROR - Connection refused for proxy 213.176.6.101:80
2024-06-04 13:49:46,450 - ERROR - Connection refused for proxy 72.10.160.92:27103
2024-06-04 13:49:46,924 - ERROR - General error for proxy 162.223.94.164:80: HTTPConnectionPool(host='162.223.94.164', port=80): Read timed out. (read timeout=5)
2024-06-04 13:49:48,482 - ERROR - General error for proxy 47.251.43.115:33333: HTTPConnectionPool(host='47.251.43.115', port=33333): Read timed out. (read timeout=5)
2024-06-04 13:49:50,172 - ERROR - General error for proxy 54.234.212.205:3128: HTTPConnectionPool(host='54.234.212.205', port=3128): Read timed out. (read timeout=5)
2024-06-04 13:49:50,560 - ERROR - Connection refused for proxy 50.223.246.226:80
2024-06-04 13:49:50,699 - ERROR - Connection refused for proxy 50.171.68.130:80
2024-06-04 13:49:50,746 - ERROR - General error for proxy 200.33.112.55:80: HTTPConnectionPool(host='200.33.112.55', port=80): Read timed out. (read timeout=5)
2024-06-04 13:49:50,918 - ERROR - Connection refused for proxy 209.146.104.56:80
2024-06-04 13:49:51,196 - ERROR - Connection refused for proxy 50.218.204.103:80
2024-06-04 13:49:51,462 - ERROR - Connection refused for proxy 50.223.239.168:80
2024-06-04 13:49:51,927 - ERROR - Connection refused for proxy 50.168.72.122:80
2024-06-04 13:49:51,989 - ERROR - Connection refused for proxy 133.18.234.13:80
2024-06-04 13:49:53,485 - ERROR - Connection refused for proxy 50.144.161.167:80
2024-06-04 13:49:55,188 - ERROR - Connection refused for proxy 50.175.212.79:80
2024-06-04 13:49:55,204 - ERROR - Connection refused for proxy 58.176.49.86:80
2024-06-04 13:49:55,576 - ERROR - Connection refused for proxy 213.143.113.82:80
2024-06-04 13:49:55,702 - ERROR - Connection refused for proxy 50.207.199.86:80
2024-06-04 13:49:55,749 - ERROR - Connection refused for proxy 50.172.75.120:80
2024-06-04 13:49:55,920 - ERROR - Connection refused for proxy 50.171.122.30:80
2024-06-04 13:49:56,469 - ERROR - Connection refused for proxy 90.84.17.133:3128
2024-06-04 13:49:56,996 - ERROR - Connection refused for proxy 38.56.70.75:999
2024-06-04 13:49:57,201 - ERROR - Connection refused for proxy 152.32.64.43:8095
2024-06-04 13:49:58,499 - ERROR - Connection refused for proxy 113.23.183.154:8989
2024-06-04 13:50:00,197 - ERROR - Connection refused for proxy 103.170.100.96:8080
2024-06-04 13:50:00,588 - ERROR - Connection refused for proxy 217.100.239.117:8090
2024-06-04 13:50:00,711 - ERROR - Connection refused for proxy 62.37.124.180:3128
2024-06-04 13:50:00,758 - ERROR - Connection refused for proxy 66.191.31.158:80
2024-06-04 13:50:00,928 - ERROR - Connection refused for proxy 213.33.126.130:80
2024-06-04 13:50:01,318 - ERROR - General error for proxy 68.183.14.206:3128: HTTPConnectionPool(host='68.183.14.206', port=3128): Read timed out. (read timeout=5)
2024-06-04 13:50:01,474 - ERROR - Connection refused for proxy 50.144.161.162:80
2024-06-04 13:50:01,877 - ERROR - Connection refused for proxy 117.250.3.58:8080
2024-06-04 13:50:02,004 - ERROR - Connection refused for proxy 50.168.72.118:80
2024-06-04 13:50:02,205 - ERROR - Connection refused for proxy 213.157.6.50:80
2024-06-04 13:50:03,356 - ERROR - Connection refused for proxy 38.54.71.67:80
2024-06-04 13:50:04,270 - ERROR - Connection refused for proxy 127.0.0.7:80
2024-06-04 13:50:05,605 - ERROR - Connection refused for proxy 50.168.72.119:80
2024-06-04 13:50:05,714 - ERROR - Connection refused for proxy 50.175.212.72:80
2024-06-04 13:50:05,760 - ERROR - Connection refused for proxy 50.217.226.45:80
2024-06-04 13:50:05,932 - ERROR - Connection refused for proxy 50.207.199.83:80
2024-06-04 13:50:06,322 - ERROR - Connection refused for proxy 50.207.199.87:80
2024-06-04 13:50:06,452 - ERROR - Connection refused for proxy 72.10.160.91:17137
2024-06-04 13:50:06,884 - ERROR - Connection refused for proxy 50.172.75.124:80
2024-06-04 13:50:07,007 - ERROR - Connection refused for proxy 85.8.68.2:80
2024-06-04 13:50:07,023 - ERROR - Connection refused for proxy 50.168.7.250:80
2024-06-04 13:50:08,360 - ERROR - Connection refused for proxy 50.168.163.179:80
2024-06-04 13:50:08,451 - ERROR - Connection refused for proxy 2024-06-04:312
2024-06-04 13:50:08,482 - ERROR - Connection refused for proxy 2024-06-03:352
2024-06-04 13:50:08,590 - ERROR - Connection refused for proxy 2024-06-02:476
2024-06-04 13:50:08,652 - ERROR - Connection refused for proxy 2024-06-01:402
2024-06-04 13:50:09,044 - ERROR - Connection refused for proxy 2024-05-31:367
2024-06-04 13:50:09,170 - ERROR - Connection refused for proxy 380:91
2024-06-04 13:50:09,605 - ERROR - Connection refused for proxy 175:42
2024-06-04 13:50:09,728 - ERROR - Connection refused for proxy 58:14
2024-06-04 13:50:12,027 - ERROR - Connection refused for proxy 50.175.212.77:80
2024-06-04 13:50:13,371 - ERROR - Connection refused for proxy 50.173.182.90:80
2024-06-04 13:50:13,464 - ERROR - Connection refused for proxy 50.230.222.202:80
2024-06-04 13:50:13,493 - ERROR - Connection refused for proxy 50.223.239.183:80
2024-06-04 13:50:13,602 - ERROR - Connection refused for proxy 50.223.239.167:80
2024-06-04 13:50:13,663 - ERROR - Connection refused for proxy 50.168.72.116:80
2024-06-04 13:50:14,052 - ERROR - Connection refused for proxy 50.222.245.44:80
2024-06-04 13:50:14,175 - ERROR - Connection refused for proxy 50.231.172.74:80
2024-06-04 13:50:14,616 - ERROR - Connection refused for proxy 50.222.245.42:80
2024-06-04 13:50:14,740 - ERROR - Connection refused for proxy 50.168.72.112:80
2024-06-04 13:50:17,030 - ERROR - Connection refused for proxy 50.222.245.43:80
2024-06-04 13:50:17,214 - ERROR - General error for proxy 209.97.150.167:8080: Exceeded 30 redirects.
2024-06-04 13:50:18,374 - ERROR - Connection refused for proxy 50.223.239.175:80
2024-06-04 13:50:18,466 - ERROR - Connection refused for proxy 50.174.7.158:80
2024-06-04 13:50:18,497 - ERROR - Connection refused for proxy 50.174.145.14:80
2024-06-04 13:50:18,606 - ERROR - Connection refused for proxy 50.222.245.50:80
2024-06-04 13:50:18,668 - ERROR - Connection refused for proxy 50.168.72.122:80
2024-06-04 13:50:19,058 - ERROR - Connection refused for proxy 50.223.239.194:80
2024-06-04 13:50:19,184 - ERROR - Connection refused for proxy 50.144.168.74:80
2024-06-04 13:50:19,745 - ERROR - Connection refused for proxy 50.222.245.45:80
2024-06-04 13:50:22,033 - ERROR - Connection refused for proxy 50.223.239.160:80
2024-06-04 13:50:22,220 - ERROR - Connection refused for proxy 50.171.177.124:80
2024-06-04 13:50:23,376 - ERROR - Connection refused for proxy 50.169.135.10:80
2024-06-04 13:50:23,469 - ERROR - Connection refused for proxy 50.222.245.46:80
2024-06-04 13:50:23,500 - ERROR - Connection refused for proxy 50.144.161.167:80
2024-06-04 13:50:23,609 - ERROR - Connection refused for proxy 50.168.163.182:80
2024-06-04 13:50:23,671 - ERROR - Connection refused for proxy 50.217.226.44:80
2024-06-04 13:50:24,061 - ERROR - Connection refused for proxy 50.175.212.66:80
2024-06-04 13:50:24,199 - ERROR - Connection refused for proxy 50.223.246.226:80
2024-06-04 13:50:24,760 - ERROR - Connection refused for proxy 50.223.239.191:80
2024-06-04 13:50:27,041 - ERROR - Connection refused for proxy 50.172.75.124:80
2024-06-04 13:50:27,230 - ERROR - Connection refused for proxy 50.174.145.9:80
2024-06-04 13:50:28,382 - ERROR - Connection refused for proxy 50.168.163.176:80
2024-06-04 13:50:28,475 - ERROR - Connection refused for proxy 50.168.72.119:80
2024-06-04 13:50:28,506 - ERROR - Connection refused for proxy 50.169.37.50:80
2024-06-04 13:50:28,616 - ERROR - Connection refused for proxy 50.239.72.19:80
2024-06-04 13:50:28,677 - ERROR - Connection refused for proxy 50.172.75.120:80
2024-06-04 13:50:29,069 - ERROR - Connection refused for proxy 50.172.75.121:80
2024-06-04 13:50:29,210 - ERROR - Connection refused for proxy 32.223.6.94:80
2024-06-04 13:50:29,772 - ERROR - Connection refused for proxy 66.191.31.158:80
2024-06-04 13:50:32,051 - ERROR - Connection refused for proxy 50.172.39.98:80
2024-06-04 13:50:32,238 - ERROR - Connection refused for proxy 50.172.75.125:80
2024-06-04 13:50:33,396 - ERROR - Connection refused for proxy 50.223.38.6:80
2024-06-04 13:50:33,490 - ERROR - Connection refused for proxy 50.174.145.13:80
2024-06-04 13:50:33,520 - ERROR - Connection refused for proxy 50.172.75.126:80
2024-06-04 13:50:33,628 - ERROR - Connection refused for proxy 50.221.230.186:80
2024-06-04 13:50:33,690 - ERROR - Connection refused for proxy 50.172.75.123:80
2024-06-04 13:50:34,082 - ERROR - Connection refused for proxy 50.174.7.162:80
2024-06-04 13:50:34,223 - ERROR - Connection refused for proxy 68.185.57.66:80
2024-06-04 13:50:34,784 - ERROR - Connection refused for proxy 50.168.163.179:80
2024-06-04 13:50:37,055 - ERROR - Connection refused for proxy 50.239.72.17:80
2024-06-04 13:50:37,241 - ERROR - Connection refused for proxy 50.174.7.154:80
2024-06-04 13:50:38,413 - ERROR - Connection refused for proxy 50.174.7.156:80
2024-06-04 13:50:38,506 - ERROR - Connection refused for proxy 50.168.163.180:80
2024-06-04 13:50:38,537 - ERROR - Connection refused for proxy 50.174.7.152:80
2024-06-04 13:50:38,631 - ERROR - Connection refused for proxy 50.168.163.178:80
2024-06-04 13:50:38,694 - ERROR - Connection refused for proxy 50.168.163.177:80
2024-06-04 13:50:39,090 - ERROR - Connection refused for proxy 50.168.72.117:80
2024-06-04 13:50:39,232 - ERROR - Connection refused for proxy 50.217.226.45:80
2024-06-04 13:50:39,426 - ERROR - Connection refused for proxy 35.185.196.38:3128
2024-06-04 13:50:39,788 - ERROR - Connection refused for proxy 50.174.7.157:80
2024-06-04 13:50:40,386 - ERROR - Connection refused for proxy 216.137.184.253:80
2024-06-04 13:50:42,064 - ERROR - Connection refused for proxy 50.217.226.43:80
2024-06-04 13:50:42,249 - ERROR - Connection refused for proxy 50.168.72.118:80
2024-06-04 13:50:43,420 - ERROR - Connection refused for proxy 50.168.72.114:80
2024-06-04 13:50:43,516 - ERROR - Connection refused for proxy 50.207.199.81:80
2024-06-04 13:50:43,701 - ERROR - Connection refused for proxy 50.207.199.82:80
2024-06-04 13:50:44,106 - ERROR - Connection refused for proxy 50.168.72.113:80
2024-06-04 13:50:44,245 - ERROR - Connection refused for proxy 50.207.199.87:80
2024-06-04 13:50:44,432 - ERROR - Connection refused for proxy 50.207.199.83:80
2024-06-04 13:50:44,791 - ERROR - Connection refused for proxy 50.168.72.115:80
2024-06-04 13:50:45,400 - ERROR - Connection refused for proxy 50.202.75.26:80
2024-06-04 13:50:47,080 - ERROR - Connection refused for proxy 50.174.7.155:80
2024-06-04 13:50:47,253 - ERROR - Connection refused for proxy 50.168.163.166:80
2024-06-04 13:50:48,436 - ERROR - Connection refused for proxy 50.217.226.46:80
2024-06-04 13:50:48,529 - ERROR - Connection refused for proxy 50.168.7.250:80
2024-06-04 13:50:48,716 - ERROR - Connection refused for proxy 50.218.204.103:80
2024-06-04 13:50:49,120 - ERROR - Connection refused for proxy 50.223.239.165:80
2024-06-04 13:50:49,260 - ERROR - Connection refused for proxy 50.145.24.186:80
2024-06-04 13:50:49,443 - ERROR - Connection refused for proxy 50.175.212.76:80
2024-06-04 13:50:49,800 - ERROR - Connection refused for proxy 50.217.226.42:80
2024-06-04 13:50:50,444 - ERROR - General error for proxy 162.223.94.166:80: HTTPConnectionPool(host='162.223.94.166', port=80): Read timed out. (read timeout=5)
2024-06-04 13:50:52,089 - ERROR - Connection refused for proxy 50.223.239.173:80
2024-06-04 13:50:52,260 - ERROR - Connection refused for proxy 50.144.161.162:80
2024-06-04 13:50:54,453 - ERROR - Connection refused for proxy 50.223.239.185:80
2024-06-04 13:50:54,468 - ERROR - Connection refused for proxy 50.223.239.166:80
2024-06-04 13:50:54,468 - ERROR - Connection refused for proxy 50.217.226.41:80
2024-06-04 13:50:54,483 - ERROR - Connection refused for proxy 50.144.166.226:80
2024-06-04 13:50:54,499 - ERROR - General error for proxy 138.68.60.8:8080: HTTPConnectionPool(host='138.68.60.8', port=8080): Read timed out. (read timeout=5)
2024-06-04 13:50:54,530 - ERROR - Connection refused for proxy 50.223.246.237:80
2024-06-04 13:50:54,809 - ERROR - Connection refused for proxy 50.222.245.40:80
2024-06-04 13:50:55,447 - ERROR - Connection refused for proxy 50.237.207.186:80
2024-06-04 13:50:57,098 - ERROR - Connection refused for proxy 50.171.187.51:80
2024-06-04 13:50:57,268 - ERROR - Connection refused for proxy 50.232.104.86:80
2024-06-04 13:50:57,286 - ERROR - General error for proxy 198.199.86.11:8080: Exceeded 30 redirects.
2024-06-04 13:50:58,151 - ERROR - Connection refused for proxy 172.96.117.205:38001
2024-06-04 13:50:59,459 - ERROR - Connection refused for proxy 50.172.75.122:80
2024-06-04 13:50:59,475 - ERROR - Connection refused for proxy 50.174.145.15:80
2024-06-04 13:50:59,475 - ERROR - Connection refused for proxy 50.175.212.74:80
2024-06-04 13:50:59,494 - ERROR - Connection refused for proxy 50.174.145.11:80
2024-06-04 13:50:59,506 - ERROR - Connection refused for proxy 50.222.245.47:80
2024-06-04 13:50:59,818 - ERROR - Connection refused for proxy 50.174.7.153:80
2024-06-04 13:51:00,462 - ERROR - Connection refused for proxy 50.239.72.18:80
2024-06-04 13:51:02,115 - ERROR - Connection refused for proxy 50.174.145.8:80
2024-06-04 13:51:02,286 - ERROR - Connection refused for proxy 50.221.74.130:80
2024-06-04 13:51:03,155 - ERROR - Connection refused for proxy 50.217.226.40:80
2024-06-04 13:51:04,467 - ERROR - Connection refused for proxy 50.168.163.181:80
2024-06-04 13:51:04,497 - ERROR - Connection refused for proxy 50.145.24.181:80
2024-06-04 13:51:04,497 - ERROR - Connection refused for proxy 50.174.7.159:80
2024-06-04 13:51:04,497 - ERROR - Connection refused for proxy 50.145.24.176:80
2024-06-04 13:51:06,322 - ERROR - Connection refused for proxy 156.239.55.151:3128
2024-06-04 13:51:07,300 - ERROR - Connection refused for proxy 156.239.55.49:3128
2024-06-04 13:51:07,487 - ERROR - Connection refused for proxy 156.239.50.176:3128
2024-06-04 13:51:09,514 - ERROR - Connection refused for proxy 104.252.131.217:3128
2024-06-04 13:51:10,525 - ERROR - Connection refused for proxy 104.165.127.165:3128
2024-06-04 13:51:10,604 - ERROR - Connection refused for proxy 156.239.49.144:3128
2024-06-04 13:51:10,744 - ERROR - Connection refused for proxy 156.239.55.41:3128
2024-06-04 13:51:12,734 - ERROR - Connection refused for proxy 156.239.48.96:3128
2024-06-04 13:51:13,146 - ERROR - Connection refused for proxy 156.239.48.128:3128
2024-06-04 13:51:14,781 - ERROR - Connection refused for proxy 104.165.169.150:3128
2024-06-04 13:51:15,751 - ERROR - Connection refused for proxy 156.239.53.22:3128
2024-06-04 13:51:16,890 - ERROR - Connection refused for proxy 104.165.127.40:3128
2024-06-04 13:51:17,342 - ERROR - Connection refused for proxy 104.164.183.176:3128
2024-06-04 13:51:17,373 - ERROR - Connection refused for proxy 104.165.169.240:3128
2024-06-04 13:51:19,951 - ERROR - Connection refused for proxy 156.239.50.148:3128
2024-06-04 13:51:20,013 - ERROR - Connection refused for proxy 104.164.183.246:3128
2024-06-04 13:51:20,402 - ERROR - Connection refused for proxy 2024-06-04:312
2024-06-04 13:51:20,432 - ERROR - Connection refused for proxy 2024-06-03:352
2024-06-04 13:51:20,760 - ERROR - Connection refused for proxy 2024-06-02:476
2024-06-04 13:51:21,794 - ERROR - Connection refused for proxy 2024-06-01:402
2024-06-04 13:51:21,918 - ERROR - Connection refused for proxy 2024-05-31:367
2024-06-04 13:51:22,151 - ERROR - Connection refused for proxy 104.252.131.170:3128
2024-06-04 13:51:22,337 - ERROR - Connection refused for proxy 380:91
2024-06-04 13:51:22,383 - ERROR - Connection refused for proxy 104.165.169.180:3128
2024-06-04 13:51:22,664 - ERROR - Connection refused for proxy 175:42
2024-06-04 13:51:22,710 - ERROR - Connection refused for proxy 58:14
2024-06-04 13:51:22,712 - INFO - ====== WebDriver manager ======
2024-06-04 13:51:22,713 - INFO - ====== WebDriver manager ======
2024-06-04 13:51:22,713 - INFO - ====== WebDriver manager ======
2024-06-04 13:51:22,714 - INFO - ====== WebDriver manager ======
2024-06-04 13:51:22,715 - INFO - ====== WebDriver manager ======
2024-06-04 13:51:23,618 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:23,618 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:23,618 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:23,618 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:23,618 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:24,045 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:24,058 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:24,058 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:24,059 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:24,103 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:24,413 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:24,429 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:24,430 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:24,448 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:24,496 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:25,213 - INFO - WebDriver version 125.0.6422.141 selected
2024-06-04 13:51:25,216 - INFO - Modern chrome version https://storage.googleapis.com/chrome-for-testing-public/125.0.6422.141/win32/chromedriver-win32.zip
2024-06-04 13:51:25,216 - INFO - About to download new driver from https://storage.googleapis.com/chrome-for-testing-public/125.0.6422.141/win32/chromedriver-win32.zip
2024-06-04 13:51:25,232 - INFO - WebDriver version 125.0.6422.141 selected
2024-06-04 13:51:25,233 - INFO - Modern chrome version https://storage.googleapis.com/chrome-for-testing-public/125.0.6422.141/win32/chromedriver-win32.zip
2024-06-04 13:51:25,233 - INFO - About to download new driver from https://storage.googleapis.com/chrome-for-testing-public/125.0.6422.141/win32/chromedriver-win32.zip
2024-06-04 13:51:25,266 - INFO - WebDriver version 125.0.6422.141 selected
2024-06-04 13:51:25,268 - INFO - Modern chrome version https://storage.googleapis.com/chrome-for-testing-public/125.0.6422.141/win32/chromedriver-win32.zip
2024-06-04 13:51:25,268 - INFO - About to download new driver from https://storage.googleapis.com/chrome-for-testing-public/125.0.6422.141/win32/chromedriver-win32.zip
2024-06-04 13:51:25,278 - INFO - WebDriver version 125.0.6422.141 selected
2024-06-04 13:51:25,281 - INFO - Modern chrome version https://storage.googleapis.com/chrome-for-testing-public/125.0.6422.141/win32/chromedriver-win32.zip
2024-06-04 13:51:25,281 - INFO - About to download new driver from https://storage.googleapis.com/chrome-for-testing-public/125.0.6422.141/win32/chromedriver-win32.zip
2024-06-04 13:51:25,297 - INFO - WebDriver version 125.0.6422.141 selected
2024-06-04 13:51:25,300 - INFO - Modern chrome version https://storage.googleapis.com/chrome-for-testing-public/125.0.6422.141/win32/chromedriver-win32.zip
2024-06-04 13:51:25,300 - INFO - About to download new driver from https://storage.googleapis.com/chrome-for-testing-public/125.0.6422.141/win32/chromedriver-win32.zip
2024-06-04 13:51:25,658 - INFO - Driver downloading response is 200
2024-06-04 13:51:25,707 - INFO - Driver downloading response is 200
2024-06-04 13:51:25,728 - INFO - Driver downloading response is 200
2024-06-04 13:51:25,737 - INFO - Driver downloading response is 200
2024-06-04 13:51:25,745 - INFO - Driver downloading response is 200
2024-06-04 13:51:26,476 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:26,749 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:26,844 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:26,847 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:26,881 - INFO - Driver has been saved in cache [C:\Users\mohan\.wdm\drivers\chromedriver\win64\125.0.6422.141]
2024-06-04 13:51:26,990 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:27,086 - INFO - Driver has been saved in cache [C:\Users\mohan\.wdm\drivers\chromedriver\win64\125.0.6422.141]
2024-06-04 13:51:27,280 - INFO - Driver has been saved in cache [C:\Users\mohan\.wdm\drivers\chromedriver\win64\125.0.6422.141]
2024-06-04 13:51:27,283 - ERROR - Request failed (retries left: 4): Message: 'chromedriver.exe' executable may have wrong permissions.

2024-06-04 13:51:27,283 - INFO - Removed invalid proxy: 18.223.25.15:80
2024-06-04 13:51:27,298 - INFO - Driver has been saved in cache [C:\Users\mohan\.wdm\drivers\chromedriver\win64\125.0.6422.141]
2024-06-04 13:51:27,312 - ERROR - Request failed (retries left: 4): [WinError 5] Access is denied: 'C:\\Users\\mohan\\.wdm\\drivers\\chromedriver\\win64\\125.0.6422.141\\chromedriver-win32\\chromedriver.exe' -> 'C:\\Users\\mohan\\.wdm\\drivers\\chromedriver\\win64\\125.0.6422.141\\chromedriver.exe'
2024-06-04 13:51:27,312 - INFO - Removed invalid proxy: 13.56.163.250:3128
2024-06-04 13:51:28,284 - INFO - ====== WebDriver manager ======
2024-06-04 13:51:28,316 - INFO - ====== WebDriver manager ======
2024-06-04 13:51:29,232 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:29,242 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:29,572 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:29,594 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:29,891 - INFO - Driver [C:\Users\mohan\.wdm\drivers\chromedriver\win64\125.0.6422.141\chromedriver-win32/chromedriver.exe] found in cache
2024-06-04 13:51:29,923 - INFO - Driver [C:\Users\mohan\.wdm\drivers\chromedriver\win64\125.0.6422.141\chromedriver-win32/chromedriver.exe] found in cache
2024-06-04 13:51:38,948 - ERROR - Request failed (retries left: 4): Search bar not found
2024-06-04 13:51:38,948 - INFO - Removed invalid proxy: 3.21.101.158:3128
2024-06-04 13:51:39,060 - ERROR - Request failed (retries left: 4): Search bar not found
2024-06-04 13:51:39,060 - INFO - Removed invalid proxy: 44.195.247.145:80
2024-06-04 13:51:39,625 - ERROR - Request failed (retries left: 4): Search bar not found
2024-06-04 13:51:39,625 - INFO - Removed invalid proxy: 8.213.129.15:1234
2024-06-04 13:51:39,949 - INFO - ====== WebDriver manager ======
2024-06-04 13:51:40,061 - INFO - ====== WebDriver manager ======
2024-06-04 13:51:40,625 - INFO - ====== WebDriver manager ======
2024-06-04 13:51:40,641 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:40,760 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:41,026 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:41,129 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:41,328 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:41,427 - INFO - Driver [C:\Users\mohan\.wdm\drivers\chromedriver\win64\125.0.6422.141\chromedriver-win32/chromedriver.exe] found in cache
2024-06-04 13:51:41,493 - ERROR - Request failed (retries left: 3): Search bar not found
2024-06-04 13:51:41,493 - INFO - Removed invalid proxy: 52.73.224.54:3128
2024-06-04 13:51:41,498 - INFO - Driver [C:\Users\mohan\.wdm\drivers\chromedriver\win64\125.0.6422.141\chromedriver-win32/chromedriver.exe] found in cache
2024-06-04 13:51:41,624 - ERROR - Request failed (retries left: 3): Search bar not found
2024-06-04 13:51:41,624 - INFO - Removed invalid proxy: 13.56.163.250:3128
2024-06-04 13:51:41,644 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:41,950 - INFO - Driver [C:\Users\mohan\.wdm\drivers\chromedriver\win64\125.0.6422.141\chromedriver-win32/chromedriver.exe] found in cache
2024-06-04 13:51:43,493 - INFO - ====== WebDriver manager ======
2024-06-04 13:51:43,624 - INFO - ====== WebDriver manager ======
2024-06-04 13:51:44,218 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:44,345 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:44,581 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:44,704 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:44,972 - INFO - Driver [C:\Users\mohan\.wdm\drivers\chromedriver\win64\125.0.6422.141\chromedriver-win32/chromedriver.exe] found in cache
2024-06-04 13:51:45,109 - INFO - Driver [C:\Users\mohan\.wdm\drivers\chromedriver\win64\125.0.6422.141\chromedriver-win32/chromedriver.exe] found in cache
2024-06-04 13:51:54,921 - ERROR - Request failed (retries left: 3): Search bar not found
2024-06-04 13:51:54,921 - INFO - Removed invalid proxy: 3.212.148.199:3128
2024-06-04 13:51:55,280 - ERROR - Request failed (retries left: 3): Search bar not found
2024-06-04 13:51:55,280 - INFO - Removed invalid proxy: 13.56.163.250:3128
2024-06-04 13:51:55,776 - ERROR - Request failed (retries left: 3): Search bar not found
2024-06-04 13:51:55,776 - INFO - Removed invalid proxy: 54.67.125.45:3128
2024-06-04 13:51:56,922 - INFO - ====== WebDriver manager ======
2024-06-04 13:51:57,281 - INFO - ====== WebDriver manager ======
2024-06-04 13:51:57,596 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:57,777 - INFO - ====== WebDriver manager ======
2024-06-04 13:51:57,941 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:57,968 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:58,292 - INFO - Driver [C:\Users\mohan\.wdm\drivers\chromedriver\win64\125.0.6422.141\chromedriver-win32/chromedriver.exe] found in cache
2024-06-04 13:51:58,324 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:58,459 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:58,554 - ERROR - Request failed (retries left: 2): Search bar not found
2024-06-04 13:51:58,554 - INFO - Removed invalid proxy: 3.12.144.146:3128
2024-06-04 13:51:58,643 - ERROR - Request failed (retries left: 2): Search bar not found
2024-06-04 13:51:58,677 - INFO - Driver [C:\Users\mohan\.wdm\drivers\chromedriver\win64\125.0.6422.141\chromedriver-win32/chromedriver.exe] found in cache
2024-06-04 13:51:58,861 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:51:59,227 - INFO - Driver [C:\Users\mohan\.wdm\drivers\chromedriver\win64\125.0.6422.141\chromedriver-win32/chromedriver.exe] found in cache
2024-06-04 13:52:02,555 - INFO - ====== WebDriver manager ======
2024-06-04 13:52:02,591 - ERROR - Request failed (retries left: 2): Message: unknown error: net::ERR_CONNECTION_RESET
  (Session info: chrome=125.0.6422.141)
Stacktrace:
	GetHandleVerifier [0x0093B8E3+45827]
	(No symbol) [0x008CDCC4]
	(No symbol) [0x007C150F]
	(No symbol) [0x007B9B2B]
	(No symbol) [0x007AEB59]
	(No symbol) [0x007AD953]
	(No symbol) [0x007AD506]
	(No symbol) [0x007AD311]
	(No symbol) [0x007AB739]
	(No symbol) [0x007ABBFB]
	(No symbol) [0x007C3CF4]
	(No symbol) [0x0083C717]
	(No symbol) [0x00822DFC]
	(No symbol) [0x0083C034]
	(No symbol) [0x00822B96]
	(No symbol) [0x007F6998]
	(No symbol) [0x007F751D]
	GetHandleVerifier [0x00BF4513+2899763]
	GetHandleVerifier [0x00C4793D+3240797]
	GetHandleVerifier [0x009C13B4+593364]
	GetHandleVerifier [0x009C82DC+621820]
	(No symbol) [0x008D70A4]
	(No symbol) [0x008D37A8]
	(No symbol) [0x008D3947]
	(No symbol) [0x008C59FE]
	BaseThreadInitThunk [0x74FB7BA9+25]
	RtlInitializeExceptionChain [0x76F4BE3B+107]
	RtlClearBits [0x76F4BDBF+191]

2024-06-04 13:52:02,591 - INFO - Removed invalid proxy: 44.219.175.186:80
2024-06-04 13:52:02,643 - INFO - ====== WebDriver manager ======
2024-06-04 13:52:03,291 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:52:03,359 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:52:03,635 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:52:03,733 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:52:04,001 - INFO - Driver [C:\Users\mohan\.wdm\drivers\chromedriver\win64\125.0.6422.141\chromedriver-win32/chromedriver.exe] found in cache
2024-06-04 13:52:04,108 - INFO - Driver [C:\Users\mohan\.wdm\drivers\chromedriver\win64\125.0.6422.141\chromedriver-win32/chromedriver.exe] found in cache
2024-06-04 13:52:06,592 - INFO - ====== WebDriver manager ======
2024-06-04 13:52:07,317 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:52:07,599 - ERROR - Request failed (retries left: 1): Message: unknown error: net::ERR_CONNECTION_RESET
  (Session info: chrome=125.0.6422.141)
Stacktrace:
	GetHandleVerifier [0x0093B8E3+45827]
	(No symbol) [0x008CDCC4]
	(No symbol) [0x007C150F]
	(No symbol) [0x007B9B2B]
	(No symbol) [0x007AEB59]
	(No symbol) [0x007AD953]
	(No symbol) [0x007AD506]
	(No symbol) [0x007AD311]
	(No symbol) [0x007AB739]
	(No symbol) [0x007ABBFB]
	(No symbol) [0x007C3CF4]
	(No symbol) [0x0083C717]
	(No symbol) [0x00822DFC]
	(No symbol) [0x0083C034]
	(No symbol) [0x00822B96]
	(No symbol) [0x007F6998]
	(No symbol) [0x007F751D]
	GetHandleVerifier [0x00BF4513+2899763]
	GetHandleVerifier [0x00C4793D+3240797]
	GetHandleVerifier [0x009C13B4+593364]
	GetHandleVerifier [0x009C82DC+621820]
	(No symbol) [0x008D70A4]
	(No symbol) [0x008D37A8]
	(No symbol) [0x008D3947]
	(No symbol) [0x008C59FE]
	BaseThreadInitThunk [0x74FB7BA9+25]
	RtlInitializeExceptionChain [0x76F4BE3B+107]
	RtlClearBits [0x76F4BDBF+191]

2024-06-04 13:52:07,599 - INFO - Removed invalid proxy: 13.59.156.167:3128
2024-06-04 13:52:07,700 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:52:08,050 - INFO - Driver [C:\Users\mohan\.wdm\drivers\chromedriver\win64\125.0.6422.141\chromedriver-win32/chromedriver.exe] found in cache
2024-06-04 13:52:11,470 - ERROR - Request failed (retries left: 1): Message: unknown error: net::ERR_CONNECTION_RESET
  (Session info: chrome=125.0.6422.141)
Stacktrace:
	GetHandleVerifier [0x0093B8E3+45827]
	(No symbol) [0x008CDCC4]
	(No symbol) [0x007C150F]
	(No symbol) [0x007B9B2B]
	(No symbol) [0x007AEB59]
	(No symbol) [0x007AD953]
	(No symbol) [0x007AD506]
	(No symbol) [0x007AD311]
	(No symbol) [0x007AB739]
	(No symbol) [0x007ABBFB]
	(No symbol) [0x007C3CF4]
	(No symbol) [0x0083C717]
	(No symbol) [0x00822DFC]
	(No symbol) [0x0083C034]
	(No symbol) [0x00822B96]
	(No symbol) [0x007F6998]
	(No symbol) [0x007F751D]
	GetHandleVerifier [0x00BF4513+2899763]
	GetHandleVerifier [0x00C4793D+3240797]
	GetHandleVerifier [0x009C13B4+593364]
	GetHandleVerifier [0x009C82DC+621820]
	(No symbol) [0x008D70A4]
	(No symbol) [0x008D37A8]
	(No symbol) [0x008D3947]
	(No symbol) [0x008C59FE]
	BaseThreadInitThunk [0x74FB7BA9+25]
	RtlInitializeExceptionChain [0x76F4BE3B+107]
	RtlClearBits [0x76F4BDBF+191]

2024-06-04 13:52:12,048 - ERROR - Request failed (retries left: 2): Search bar not found
2024-06-04 13:52:12,048 - INFO - Removed invalid proxy: 13.56.192.187:80
2024-06-04 13:52:12,713 - ERROR - Request failed (retries left: 2): Search bar not found
2024-06-04 13:52:15,600 - INFO - ====== WebDriver manager ======
2024-06-04 13:52:16,048 - INFO - ====== WebDriver manager ======
2024-06-04 13:52:16,264 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:52:16,546 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:52:16,711 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:52:16,713 - INFO - ====== WebDriver manager ======
2024-06-04 13:52:16,871 - INFO - Driver [C:\Users\mohan\.wdm\drivers\chromedriver\win64\125.0.6422.141\chromedriver-win32/chromedriver.exe] found in cache
2024-06-04 13:52:17,010 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:52:17,305 - INFO - Driver [C:\Users\mohan\.wdm\drivers\chromedriver\win64\125.0.6422.141\chromedriver-win32/chromedriver.exe] found in cache
2024-06-04 13:52:17,366 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:52:17,691 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:52:18,007 - INFO - Driver [C:\Users\mohan\.wdm\drivers\chromedriver\win64\125.0.6422.141\chromedriver-win32/chromedriver.exe] found in cache
2024-06-04 13:52:19,470 - INFO - ====== WebDriver manager ======
2024-06-04 13:52:20,294 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:52:20,697 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:52:21,117 - INFO - Driver [C:\Users\mohan\.wdm\drivers\chromedriver\win64\125.0.6422.141\chromedriver-win32/chromedriver.exe] found in cache
2024-06-04 13:52:21,555 - ERROR - Request failed (retries left: 1): Search bar not found
2024-06-04 13:52:26,493 - INFO - Fetched HTML content for page 4
2024-06-04 13:52:26,501 - WARNING - No search results found on the page.
2024-06-04 13:52:26,602 - INFO - Fetched HTML content for page 3
2024-06-04 13:52:26,610 - WARNING - No search results found on the page.
2024-06-04 13:52:27,568 - INFO - Fetched HTML content for page 2
2024-06-04 13:52:27,576 - WARNING - No search results found on the page.
2024-06-04 13:52:29,556 - INFO - ====== WebDriver manager ======
2024-06-04 13:52:30,273 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:52:30,585 - INFO - Get LATEST chromedriver version for google-chrome
2024-06-04 13:52:30,600 - INFO - Fetched results: []
2024-06-04 13:52:30,709 - INFO - Fetched results: []
2024-06-04 13:52:30,895 - INFO - Driver [C:\Users\mohan\.wdm\drivers\chromedriver\win64\125.0.6422.141\chromedriver-win32/chromedriver.exe] found in cache
2024-06-04 13:52:31,671 - INFO - Fetched results: []
2024-06-04 13:52:31,875 - INFO - Fetched HTML content for page 1
2024-06-04 13:52:31,883 - WARNING - No search results found on the page.
2024-06-04 13:52:35,967 - INFO - Fetched results: []
2024-06-04 13:52:41,240 - INFO - Fetched HTML content for page 0
2024-06-04 13:52:41,248 - WARNING - No search results found on the page.
2024-06-04 13:52:45,327 - INFO - Fetched results: []
2024-06-04 13:52:45,328 - INFO - Fetched results: []
2024-06-04 13:52:45,328 - WARNING - No results fetched.
2024-06-04 13:52:45,330 - INFO - Results written to CSV: 2024-06-04_results.csv
2024-06-04 13:53:26,239 - INFO - Starting to fetch search results for query: This work was supported in part through the NYU IT High Performance Computing resources, services, and staff expertise from page 0
2024-06-04 13:53:26,627 - WARNING - Row does not contain enough cells
2024-06-04 13:53:27,038 - WARNING - Row does not contain enough cells
2024-06-04 13:53:27,634 - ERROR - Proxy error for proxy 203.205.9.105:8080: HTTPConnectionPool(host='203.205.9.105', port=8080): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', RemoteDisconnected('Remote end closed connection without response')))
2024-06-04 13:53:27,873 - ERROR - Proxy error for proxy 117.250.3.58:8080: ('Connection broken: IncompleteRead(3194 bytes read, 5920 more expected)', IncompleteRead(3194 bytes read, 5920 more expected))
2024-06-04 13:53:29,237 - ERROR - Proxy error for proxy 72.10.160.91:17137: HTTPConnectionPool(host='72.10.160.91', port=17137): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001CB2A2DF110>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')))
2024-06-04 13:53:29,727 - ERROR - Proxy error for proxy 67.43.228.253:1503: HTTPConnectionPool(host='67.43.228.253', port=1503): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001CB2A2DC490>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')))
2024-06-04 13:53:30,883 - ERROR - Proxy error for proxy 72.10.160.172:16679: HTTPConnectionPool(host='72.10.160.172', port=16679): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001CB2940D910>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')))
2024-06-04 13:53:31,871 - ERROR - Proxy error for proxy 67.43.228.251:2081: HTTPConnectionPool(host='67.43.228.251', port=2081): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001CB293FD410>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')))
2024-06-04 13:53:32,052 - ERROR - Proxy error for proxy 51.145.176.250:8080: HTTPConnectionPool(host='51.145.176.250', port=8080): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2F1D50>, 'Connection to 51.145.176.250 timed out. (connect timeout=5)')))
2024-06-04 13:53:32,052 - ERROR - Proxy error for proxy 185.217.136.67:1337: HTTPConnectionPool(host='185.217.136.67', port=1337): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2E5690>, 'Connection to 185.217.136.67 timed out. (connect timeout=5)')))
2024-06-04 13:53:32,455 - ERROR - Proxy error for proxy 171.251.92.87:5404: HTTPConnectionPool(host='171.251.92.87', port=5404): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2D4310>, 'Connection to 171.251.92.87 timed out. (connect timeout=5)')))
2024-06-04 13:53:32,490 - ERROR - Proxy error for proxy 61.97.191.101:9070: HTTPConnectionPool(host='61.97.191.101', port=9070): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', RemoteDisconnected('Remote end closed connection without response')))
2024-06-04 13:53:32,531 - ERROR - Proxy error for proxy 36.111.143.8:15777: HTTPConnectionPool(host='36.111.143.8', port=15777): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None)))
2024-06-04 13:53:32,641 - ERROR - Proxy error for proxy 130.245.128.193:8080: HTTPConnectionPool(host='130.245.128.193', port=8080): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2941B790>, 'Connection to 130.245.128.193 timed out. (connect timeout=5)')))
2024-06-04 13:53:32,877 - ERROR - Proxy error for proxy 41.65.251.47:1981: HTTPConnectionPool(host='41.65.251.47', port=1981): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2DF410>, 'Connection to 41.65.251.47 timed out. (connect timeout=5)')))
2024-06-04 13:53:33,328 - ERROR - Proxy error for proxy 8.219.97.248:80: HTTPConnectionPool(host='8.219.97.248', port=80): Read timed out. (read timeout=5)
2024-06-04 13:53:34,244 - ERROR - Proxy error for proxy 34.92.250.88:11111: HTTPConnectionPool(host='34.92.250.88', port=11111): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2940D010>, 'Connection to 34.92.250.88 timed out. (connect timeout=5)')))
2024-06-04 13:53:34,668 - ERROR - Proxy error for proxy 67.43.236.18:15267: HTTPConnectionPool(host='67.43.236.18', port=15267): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001CB2A2C7C90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')))
2024-06-04 13:53:34,777 - ERROR - Proxy error for proxy 72.10.160.170:33147: HTTPConnectionPool(host='72.10.160.170', port=33147): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001CB2A2F3390>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')))
2024-06-04 13:53:35,039 - ERROR - Proxy error for proxy 67.43.228.252:3187: HTTPConnectionPool(host='67.43.228.252', port=3187): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001CB2A2E4950>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')))
2024-06-04 13:53:36,383 - ERROR - Proxy error for proxy 67.43.227.227:17629: HTTPConnectionPool(host='67.43.227.227', port=17629): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001CB2A2D7590>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')))
2024-06-04 13:53:36,884 - ERROR - Proxy error for proxy 66.188.181.143:8080: HTTPConnectionPool(host='66.188.181.143', port=8080): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2D7310>, 'Connection to 66.188.181.143 timed out. (connect timeout=5)')))
2024-06-04 13:53:36,994 - ERROR - Proxy error for proxy 203.189.88.156:80: HTTPConnectionPool(host='203.189.88.156', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB29ECF750>, 'Connection to 203.189.88.156 timed out. (connect timeout=5)')))
2024-06-04 13:53:37,460 - ERROR - Proxy error for proxy 54.67.45.218:3128: HTTPConnectionPool(host='54.67.45.218', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293F6A50>, 'Connection to 54.67.45.218 timed out. (connect timeout=5)')))
2024-06-04 13:53:37,492 - ERROR - Proxy error for proxy 34.154.161.152:80: HTTPConnectionPool(host='34.154.161.152', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293F7F90>, 'Connection to 34.154.161.152 timed out. (connect timeout=5)')))
2024-06-04 13:53:37,628 - ERROR - Proxy error for proxy 20.44.190.150:3129: HTTPConnectionPool(host='20.44.190.150', port=3129): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001CB2941CB90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')))
2024-06-04 13:53:37,863 - ERROR - Proxy error for proxy 20.204.212.76:3129: HTTPConnectionPool(host='20.204.212.76', port=3129): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001CB2940E310>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')))
2024-06-04 13:53:38,102 - ERROR - Proxy error for proxy 20.204.214.79:3129: HTTPConnectionPool(host='20.204.214.79', port=3129): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001CB2A2E4C50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')))
2024-06-04 13:53:38,361 - ERROR - Proxy error for proxy 45.77.147.46:3128: HTTPConnectionPool(host='45.77.147.46', port=3128): Read timed out. (read timeout=5)
2024-06-04 13:53:38,933 - ERROR - Proxy error for proxy 72.10.160.90:20931: HTTPConnectionPool(host='72.10.160.90', port=20931): Read timed out. (read timeout=5)
2024-06-04 13:53:39,953 - ERROR - Proxy error for proxy 20.44.188.17:3129: HTTPConnectionPool(host='20.44.188.17', port=3129): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001CB2A2C6C50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')))
2024-06-04 13:53:40,019 - ERROR - Proxy error for proxy 20.204.212.45:3129: HTTPConnectionPool(host='20.204.212.45', port=3129): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001CB29ECA4D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')))
2024-06-04 13:53:40,542 - ERROR - Proxy error for proxy 20.219.176.57:3129: HTTPConnectionPool(host='20.219.176.57', port=3129): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001CB2A2D6B50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')))
2024-06-04 13:53:42,639 - ERROR - Proxy error for proxy 80.87.178.175:8080: HTTPConnectionPool(host='80.87.178.175', port=8080): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2DE690>, 'Connection to 80.87.178.175 timed out. (connect timeout=5)')))
2024-06-04 13:53:42,872 - ERROR - Proxy error for proxy 161.34.39.153:3128: HTTPConnectionPool(host='161.34.39.153', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2F0190>, 'Connection to 161.34.39.153 timed out. (connect timeout=5)')))
2024-06-04 13:53:42,872 - ERROR - Proxy error for proxy 188.247.194.210:3128: HTTPConnectionPool(host='188.247.194.210', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001CB2941C850>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')))
2024-06-04 13:53:43,369 - ERROR - Proxy error for proxy 103.139.127.243:8080: HTTPConnectionPool(host='103.139.127.243', port=8080): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB26FA9C50>, 'Connection to 103.139.127.243 timed out. (connect timeout=5)')))
2024-06-04 13:53:43,650 - ERROR - Proxy error for proxy 154.91.1.213:3129: HTTPConnectionPool(host='154.91.1.213', port=3129): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2E6E50>, 'Connection to 154.91.1.213 timed out. (connect timeout=5)')))
2024-06-04 13:53:43,660 - ERROR - Proxy error for proxy 221.168.33.155:8080: HTTPConnectionPool(host='221.168.33.155', port=8080): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None)))
2024-06-04 13:53:44,458 - ERROR - Proxy error for proxy 50.28.7.107:80: HTTPConnectionPool(host='50.28.7.107', port=80): Read timed out. (read timeout=5)
2024-06-04 13:53:45,233 - ERROR - Proxy error for proxy 111.206.0.99:8181: HTTPConnectionPool(host='111.206.0.99', port=8181): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None)))
2024-06-04 13:53:45,560 - ERROR - Proxy error for proxy 188.132.222.37:8080: HTTPConnectionPool(host='188.132.222.37', port=8080): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2D5490>, 'Connection to 188.132.222.37 timed out. (connect timeout=5)')))
2024-06-04 13:53:47,708 - ERROR - Proxy error for proxy 189.240.60.164:9090: HTTPConnectionPool(host='189.240.60.164', port=9090): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A302790>, 'Connection to 189.240.60.164 timed out. (connect timeout=5)')))
2024-06-04 13:53:48,175 - ERROR - Proxy error for proxy 62.33.53.248:3128: HTTPConnectionPool(host='62.33.53.248', port=3128): Read timed out. (read timeout=5)
2024-06-04 13:53:48,378 - ERROR - Proxy error for proxy 31.45.237.146:8080: HTTPConnectionPool(host='31.45.237.146', port=8080): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2D4FD0>, 'Connection to 31.45.237.146 timed out. (connect timeout=5)')))
2024-06-04 13:53:48,673 - ERROR - Proxy error for proxy 103.247.122.166:8080: HTTPConnectionPool(host='103.247.122.166', port=8080): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293FE690>, 'Connection to 103.247.122.166 timed out. (connect timeout=5)')))
2024-06-04 13:53:49,045 - ERROR - Proxy error for proxy 186.96.50.113:999: HTTPConnectionPool(host='186.96.50.113', port=999): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2DE350>, 'Connection to 186.96.50.113 timed out. (connect timeout=5)')))
2024-06-04 13:53:50,242 - ERROR - Proxy error for proxy 191.243.46.166:43241: HTTPConnectionPool(host='191.243.46.166', port=43241): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293A1C50>, 'Connection to 191.243.46.166 timed out. (connect timeout=5)')))
2024-06-04 13:53:50,538 - ERROR - Proxy error for proxy 72.10.160.94:3901: HTTPConnectionPool(host='72.10.160.94', port=3901): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001CB27074510>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')))
2024-06-04 13:53:51,239 - ERROR - Proxy error for proxy 20.204.214.23:3129: HTTPConnectionPool(host='20.204.214.23', port=3129): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001CB2940D010>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')))
2024-06-04 13:53:51,579 - ERROR - Proxy error for proxy 109.164.38.189:2306: HTTPConnectionPool(host='109.164.38.189', port=2306): Max retries exceeded with url: http://www.google.com/sorry/index?continue=http://www.google.com/&q=EgRtpHtSGKip_bIGIikGc9UCIw7K4cuNGqT03fElLTcol1P3trdxVP3W7YEwnNY9OL_xD6Ak0zIGPmpjbmRyWgFD (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2E4190>, 'Connection to 109.164.38.189 timed out. (connect timeout=5)')))
2024-06-04 13:53:51,717 - ERROR - Proxy error for proxy 149.28.128.233:10000: HTTPConnectionPool(host='149.28.128.233', port=10000): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2DC390>, 'Connection to 149.28.128.233 timed out. (connect timeout=5)')))
2024-06-04 13:53:51,747 - ERROR - Proxy error for proxy 185.191.236.162:3128: HTTPConnectionPool(host='185.191.236.162', port=3128): Read timed out. (read timeout=5)
2024-06-04 13:53:52,415 - ERROR - Proxy error for proxy 67.43.227.230:5097: HTTPConnectionPool(host='67.43.227.230', port=5097): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001CB2A2C4B10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')))
2024-06-04 13:53:52,653 - ERROR - Proxy error for proxy 67.43.236.22:18891: HTTPConnectionPool(host='67.43.236.22', port=18891): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001CB2A2C7750>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')))
2024-06-04 13:53:53,701 - ERROR - Proxy error for proxy 154.127.240.120:64002: HTTPConnectionPool(host='154.127.240.120', port=64002): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293FC390>, 'Connection to 154.127.240.120 timed out. (connect timeout=5)')))
2024-06-04 13:53:53,794 - ERROR - Proxy error for proxy 38.65.172.10:999: HTTPConnectionPool(host='38.65.172.10', port=999): Read timed out. (read timeout=5)
2024-06-04 13:53:53,858 - ERROR - Proxy error for proxy 146.56.154.83:21000: HTTPConnectionPool(host='146.56.154.83', port=21000): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None)))
2024-06-04 13:53:56,586 - ERROR - Proxy error for proxy 154.127.240.119:64006: HTTPConnectionPool(host='154.127.240.119', port=64006): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB29295110>, 'Connection to 154.127.240.119 timed out. (connect timeout=5)')))
2024-06-04 13:53:56,638 - ERROR - Proxy error for proxy 154.21.201.49:8888: HTTPConnectionPool(host='154.21.201.49', port=8888): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001CB2940EE90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')))
2024-06-04 13:53:56,694 - ERROR - Proxy error for proxy 49.254.107.184:28766: HTTPConnectionPool(host='49.254.107.184', port=28766): Read timed out. (read timeout=5)
2024-06-04 13:53:56,726 - ERROR - Proxy error for proxy 154.127.240.124:64001: HTTPConnectionPool(host='154.127.240.124', port=64001): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293A26D0>, 'Connection to 154.127.240.124 timed out. (connect timeout=5)')))
2024-06-04 13:53:56,757 - ERROR - Proxy error for proxy 154.127.240.114:64002: HTTPConnectionPool(host='154.127.240.114', port=64002): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2940FB90>, 'Connection to 154.127.240.114 timed out. (connect timeout=5)')))
2024-06-04 13:53:57,428 - ERROR - Proxy error for proxy 154.127.240.115:64001: HTTPConnectionPool(host='154.127.240.115', port=64001): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2D4E90>, 'Connection to 154.127.240.115 timed out. (connect timeout=5)')))
2024-06-04 13:53:57,661 - ERROR - Proxy error for proxy 154.127.240.117:64001: HTTPConnectionPool(host='154.127.240.117', port=64001): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2DC990>, 'Connection to 154.127.240.117 timed out. (connect timeout=5)')))
2024-06-04 13:53:58,716 - ERROR - Proxy error for proxy 114.129.2.82:8081: HTTPConnectionPool(host='114.129.2.82', port=8081): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2705CFD0>, 'Connection to 114.129.2.82 timed out. (connect timeout=5)')))
2024-06-04 13:53:58,796 - ERROR - Proxy error for proxy 20.27.86.185:8080: HTTPConnectionPool(host='20.27.86.185', port=8080): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2941B0D0>, 'Connection to 20.27.86.185 timed out. (connect timeout=5)')))
2024-06-04 13:53:58,873 - ERROR - Proxy error for proxy 148.72.165.7:10529: HTTPConnectionPool(host='148.72.165.7', port=10529): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293A0390>, 'Connection to 148.72.165.7 timed out. (connect timeout=5)')))
2024-06-04 13:53:59,255 - ERROR - Proxy error for proxy 87.247.186.40:1080: HTTPConnectionPool(host='87.247.186.40', port=1080): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', RemoteDisconnected('Remote end closed connection without response')))
2024-06-04 13:54:01,589 - ERROR - Proxy error for proxy 199.168.175.179:80: HTTPConnectionPool(host='199.168.175.179', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2C60D0>, 'Connection to 199.168.175.179 timed out. (connect timeout=5)')))
2024-06-04 13:54:01,698 - ERROR - Proxy error for proxy 43.133.59.220:3128: HTTPConnectionPool(host='43.133.59.220', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2705F890>, 'Connection to 43.133.59.220 timed out. (connect timeout=5)')))
2024-06-04 13:54:01,729 - ERROR - Proxy error for proxy 91.108.130.111:32650: HTTPConnectionPool(host='91.108.130.111', port=32650): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2705D710>, 'Connection to 91.108.130.111 timed out. (connect timeout=5)')))
2024-06-04 13:54:01,760 - ERROR - Proxy error for proxy 66.70.235.23:5454: HTTPConnectionPool(host='66.70.235.23', port=5454): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293A3CD0>, 'Connection to 66.70.235.23 timed out. (connect timeout=5)')))
2024-06-04 13:54:02,065 - ERROR - Proxy error for proxy 124.70.78.157:8000: HTTPConnectionPool(host='124.70.78.157', port=8000): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None)))
2024-06-04 13:54:02,433 - ERROR - Proxy error for proxy 94.142.137.203:8080: HTTPConnectionPool(host='94.142.137.203', port=8080): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2F0890>, 'Connection to 94.142.137.203 timed out. (connect timeout=5)')))
2024-06-04 13:54:02,668 - ERROR - Proxy error for proxy 149.28.147.110:10000: HTTPConnectionPool(host='149.28.147.110', port=10000): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB27074610>, 'Connection to 149.28.147.110 timed out. (connect timeout=5)')))
2024-06-04 13:54:02,915 - ERROR - Proxy error for proxy 43.134.32.184:3128: HTTPConnectionPool(host='43.134.32.184', port=3128): Read timed out. (read timeout=5)
2024-06-04 13:54:03,723 - ERROR - Proxy error for proxy 84.47.161.3:8118: HTTPConnectionPool(host='84.47.161.3', port=8118): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2D76D0>, 'Connection to 84.47.161.3 timed out. (connect timeout=5)')))
2024-06-04 13:54:03,878 - ERROR - Proxy error for proxy 182.253.140.250:8080: HTTPConnectionPool(host='182.253.140.250', port=8080): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB27074590>, 'Connection to 182.253.140.250 timed out. (connect timeout=5)')))
2024-06-04 13:54:04,266 - ERROR - Proxy error for proxy 8.213.129.15:1234: HTTPConnectionPool(host='8.213.129.15', port=1234): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2D5C90>, 'Connection to 8.213.129.15 timed out. (connect timeout=5)')))
2024-06-04 13:54:05,628 - ERROR - Proxy error for proxy 2024-06-04:312: HTTPConnectionPool(host='2024-06-04', port=312): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NameResolutionError("<urllib3.connection.HTTPConnection object at 0x000001CB29E7BC90>: Failed to resolve '2024-06-04' ([Errno 11001] getaddrinfo failed)")))
2024-06-04 13:54:06,438 - ERROR - Proxy error for proxy 2024-06-03:352: HTTPConnectionPool(host='2024-06-03', port=352): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NameResolutionError("<urllib3.connection.HTTPConnection object at 0x000001CB293A2410>: Failed to resolve '2024-06-03' ([Errno 11001] getaddrinfo failed)")))
2024-06-04 13:54:06,592 - ERROR - Proxy error for proxy 2024-06-02:476: HTTPConnectionPool(host='2024-06-02', port=476): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NameResolutionError("<urllib3.connection.HTTPConnection object at 0x000001CB2940F790>: Failed to resolve '2024-06-02' ([Errno 11001] getaddrinfo failed)")))
2024-06-04 13:54:06,702 - ERROR - Proxy error for proxy 47.254.90.125:8800: HTTPConnectionPool(host='47.254.90.125', port=8800): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2705F5D0>, 'Connection to 47.254.90.125 timed out. (connect timeout=5)')))
2024-06-04 13:54:06,734 - ERROR - Proxy error for proxy 103.168.44.210:3127: HTTPConnectionPool(host='103.168.44.210', port=3127): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2E4F90>, 'Connection to 103.168.44.210 timed out. (connect timeout=5)')))
2024-06-04 13:54:06,765 - ERROR - Proxy error for proxy 186.125.218.188:999: HTTPConnectionPool(host='186.125.218.188', port=999): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB29ECF7D0>, 'Connection to 186.125.218.188 timed out. (connect timeout=5)')))
2024-06-04 13:54:06,982 - ERROR - Proxy error for proxy 2024-06-01:402: HTTPConnectionPool(host='2024-06-01', port=402): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NameResolutionError("<urllib3.connection.HTTPConnection object at 0x000001CB2A2F32D0>: Failed to resolve '2024-06-01' ([Errno 11001] getaddrinfo failed)")))
2024-06-04 13:54:07,076 - ERROR - Proxy error for proxy 181.119.84.229:999: HTTPConnectionPool(host='181.119.84.229', port=999): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2705D210>, 'Connection to 181.119.84.229 timed out. (connect timeout=5)')))
2024-06-04 13:54:07,450 - ERROR - Proxy error for proxy 118.99.119.11:8080: HTTPConnectionPool(host='118.99.119.11', port=8080): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2941CC10>, 'Connection to 118.99.119.11 timed out. (connect timeout=5)')))
2024-06-04 13:54:07,683 - ERROR - Proxy error for proxy 202.93.244.38:8080: HTTPConnectionPool(host='202.93.244.38', port=8080): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2D6ED0>, 'Connection to 202.93.244.38 timed out. (connect timeout=5)')))
2024-06-04 13:54:08,352 - ERROR - Proxy error for proxy 2024-05-31:367: HTTPConnectionPool(host='2024-05-31', port=367): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NameResolutionError("<urllib3.connection.HTTPConnection object at 0x000001CB2A303250>: Failed to resolve '2024-05-31' ([Errno 11001] getaddrinfo failed)")))
2024-06-04 13:54:09,160 - ERROR - Proxy error for proxy 380:91: HTTPConnectionPool(host='380', port=91): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NameResolutionError("<urllib3.connection.HTTPConnection object at 0x000001CB2A2D4B50>: Failed to resolve '380' ([Errno 11001] getaddrinfo failed)")))
2024-06-04 13:54:09,314 - ERROR - Proxy error for proxy 175:42: HTTPConnectionPool(host='175', port=42): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NameResolutionError("<urllib3.connection.HTTPConnection object at 0x000001CB293A3B90>: Failed to resolve '175' ([Errno 11001] getaddrinfo failed)")))
2024-06-04 13:54:09,422 - ERROR - Proxy error for proxy 58:14: HTTPConnectionPool(host='58', port=14): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NameResolutionError("<urllib3.connection.HTTPConnection object at 0x000001CB2705DB90>: Failed to resolve '58' ([Errno 11001] getaddrinfo failed)")))
2024-06-04 13:54:11,750 - ERROR - Proxy error for proxy 50.174.145.9:80: HTTPConnectionPool(host='50.174.145.9', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A321410>, 'Connection to 50.174.145.9 timed out. (connect timeout=5)')))
2024-06-04 13:54:11,781 - ERROR - Proxy error for proxy 50.172.75.125:80: HTTPConnectionPool(host='50.172.75.125', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2D6250>, 'Connection to 50.172.75.125 timed out. (connect timeout=5)')))
2024-06-04 13:54:11,983 - ERROR - Proxy error for proxy 50.168.163.182:80: HTTPConnectionPool(host='50.168.163.182', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2940E3D0>, 'Connection to 50.168.163.182 timed out. (connect timeout=5)')))
2024-06-04 13:54:12,092 - ERROR - Proxy error for proxy 50.232.104.86:80: HTTPConnectionPool(host='50.232.104.86', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2940DA10>, 'Connection to 50.232.104.86 timed out. (connect timeout=5)')))
2024-06-04 13:54:12,464 - ERROR - Proxy error for proxy 83.1.176.118:80: HTTPConnectionPool(host='83.1.176.118', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB27041850>, 'Connection to 83.1.176.118 timed out. (connect timeout=5)')))
2024-06-04 13:54:12,698 - ERROR - Proxy error for proxy 82.119.96.254:80: HTTPConnectionPool(host='82.119.96.254', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2C6E90>, 'Connection to 82.119.96.254 timed out. (connect timeout=5)')))
2024-06-04 13:54:13,367 - ERROR - Proxy error for proxy 50.217.226.46:80: HTTPConnectionPool(host='50.217.226.46', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2940CA50>, 'Connection to 50.217.226.46 timed out. (connect timeout=5)')))
2024-06-04 13:54:14,439 - ERROR - Proxy error for proxy 50.145.24.176:80: HTTPConnectionPool(host='50.145.24.176', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2705C9D0>, 'Connection to 50.145.24.176 timed out. (connect timeout=5)')))
2024-06-04 13:54:14,733 - ERROR - Proxy error for proxy 50.217.226.43:80: HTTPConnectionPool(host='50.217.226.43', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB29ECF250>, 'Connection to 50.217.226.43 timed out. (connect timeout=5)')))
2024-06-04 13:54:15,302 - ERROR - Proxy error for proxy 50.223.239.175:80: HTTPConnectionPool(host='50.223.239.175', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293F6490>, 'Connection to 50.223.239.175 timed out. (connect timeout=5)')))
2024-06-04 13:54:16,763 - ERROR - Proxy error for proxy 50.145.24.186:80: HTTPConnectionPool(host='50.145.24.186', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB29418290>, 'Connection to 50.145.24.186 timed out. (connect timeout=5)')))
2024-06-04 13:54:16,795 - ERROR - Proxy error for proxy 50.217.226.42:80: HTTPConnectionPool(host='50.217.226.42', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2940CF50>, 'Connection to 50.217.226.42 timed out. (connect timeout=5)')))
2024-06-04 13:54:16,998 - ERROR - Proxy error for proxy 50.175.212.77:80: HTTPConnectionPool(host='50.175.212.77', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2D6410>, 'Connection to 50.175.212.77 timed out. (connect timeout=5)')))
2024-06-04 13:54:17,107 - ERROR - Proxy error for proxy 50.172.75.121:80: HTTPConnectionPool(host='50.172.75.121', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2C4E50>, 'Connection to 50.172.75.121 timed out. (connect timeout=5)')))
2024-06-04 13:54:17,477 - ERROR - Proxy error for proxy 50.168.163.181:80: HTTPConnectionPool(host='50.168.163.181', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A3761D0>, 'Connection to 50.168.163.181 timed out. (connect timeout=5)')))
2024-06-04 13:54:17,711 - ERROR - Proxy error for proxy 50.168.72.112:80: HTTPConnectionPool(host='50.168.72.112', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB27043490>, 'Connection to 50.168.72.112 timed out. (connect timeout=5)')))
2024-06-04 13:54:18,377 - ERROR - Proxy error for proxy 50.222.245.44:80: HTTPConnectionPool(host='50.222.245.44', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2E7390>, 'Connection to 50.222.245.44 timed out. (connect timeout=5)')))
2024-06-04 13:54:19,746 - ERROR - Proxy error for proxy 50.168.72.114:80: HTTPConnectionPool(host='50.168.72.114', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2940D1D0>, 'Connection to 50.168.72.114 timed out. (connect timeout=5)')))
2024-06-04 13:54:20,319 - ERROR - Proxy error for proxy 41.207.187.178:80: HTTPConnectionPool(host='41.207.187.178', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293EBD10>, 'Connection to 41.207.187.178 timed out. (connect timeout=5)')))
2024-06-04 13:54:20,613 - ERROR - Proxy error for proxy 50.174.7.157:80: HTTPConnectionPool(host='50.174.7.157', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2705E410>, 'Connection to 50.174.7.157 timed out. (connect timeout=5)')))
2024-06-04 13:54:21,774 - ERROR - Proxy error for proxy 50.144.166.226:80: HTTPConnectionPool(host='50.144.166.226', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293FF190>, 'Connection to 50.144.166.226 timed out. (connect timeout=5)')))
2024-06-04 13:54:21,805 - ERROR - Proxy error for proxy 190.58.248.86:80: HTTPConnectionPool(host='190.58.248.86', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A303110>, 'Connection to 190.58.248.86 timed out. (connect timeout=5)')))
2024-06-04 13:54:22,009 - ERROR - Proxy error for proxy 50.223.239.194:80: HTTPConnectionPool(host='50.223.239.194', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2705D550>, 'Connection to 50.223.239.194 timed out. (connect timeout=5)')))
2024-06-04 13:54:22,119 - ERROR - Proxy error for proxy 50.175.212.74:80: HTTPConnectionPool(host='50.175.212.74', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB270752D0>, 'Connection to 50.175.212.74 timed out. (connect timeout=5)')))
2024-06-04 13:54:22,493 - ERROR - Proxy error for proxy 50.144.168.74:80: HTTPConnectionPool(host='50.144.168.74', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293FC5D0>, 'Connection to 50.144.168.74 timed out. (connect timeout=5)')))
2024-06-04 13:54:22,724 - ERROR - Proxy error for proxy 50.217.226.41:80: HTTPConnectionPool(host='50.217.226.41', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2D74D0>, 'Connection to 50.217.226.41 timed out. (connect timeout=5)')))
2024-06-04 13:54:23,380 - ERROR - Proxy error for proxy 50.223.239.185:80: HTTPConnectionPool(host='50.223.239.185', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB292036D0>, 'Connection to 50.223.239.185 timed out. (connect timeout=5)')))
2024-06-04 13:54:24,763 - ERROR - Proxy error for proxy 50.172.75.122:80: HTTPConnectionPool(host='50.172.75.122', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2E6550>, 'Connection to 50.172.75.122 timed out. (connect timeout=5)')))
2024-06-04 13:54:25,616 - ERROR - Proxy error for proxy 50.173.182.90:80: HTTPConnectionPool(host='50.173.182.90', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2D6D50>, 'Connection to 50.173.182.90 timed out. (connect timeout=5)')))
2024-06-04 13:54:25,852 - ERROR - Proxy error for proxy 50.222.245.50:80: HTTPConnectionPool(host='50.222.245.50', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2705C2D0>, 'Connection to 50.222.245.50 timed out. (connect timeout=5)')))
2024-06-04 13:54:26,814 - ERROR - Proxy error for proxy 50.223.38.6:80: HTTPConnectionPool(host='50.223.38.6', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2F3C50>, 'Connection to 50.223.38.6 timed out. (connect timeout=5)')))
2024-06-04 13:54:27,016 - ERROR - Proxy error for proxy 50.168.72.117:80: HTTPConnectionPool(host='50.168.72.117', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB29419ED0>, 'Connection to 50.168.72.117 timed out. (connect timeout=5)')))
2024-06-04 13:54:27,124 - ERROR - Proxy error for proxy 50.174.145.11:80: HTTPConnectionPool(host='50.174.145.11', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2C7810>, 'Connection to 50.174.145.11 timed out. (connect timeout=5)')))
2024-06-04 13:54:27,498 - ERROR - Proxy error for proxy 50.222.245.42:80: HTTPConnectionPool(host='50.222.245.42', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2E7C50>, 'Connection to 50.222.245.42 timed out. (connect timeout=5)')))
2024-06-04 13:54:27,734 - ERROR - Proxy error for proxy 50.222.245.47:80: HTTPConnectionPool(host='50.222.245.47', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A3032D0>, 'Connection to 50.222.245.47 timed out. (connect timeout=5)')))
2024-06-04 13:54:28,170 - ERROR - Proxy error for proxy 50.221.230.186:80: HTTPConnectionPool(host='50.221.230.186', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293F5910>, 'Connection to 50.221.230.186 timed out. (connect timeout=5)')))
2024-06-04 13:54:28,389 - ERROR - Proxy error for proxy 50.169.135.10:80: HTTPConnectionPool(host='50.169.135.10', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2940D1D0>, 'Connection to 50.169.135.10 timed out. (connect timeout=5)')))
2024-06-04 13:54:29,777 - ERROR - Proxy error for proxy 50.172.75.126:80: HTTPConnectionPool(host='50.172.75.126', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293F6BD0>, 'Connection to 50.172.75.126 timed out. (connect timeout=5)')))
2024-06-04 13:54:30,633 - ERROR - Proxy error for proxy 50.230.222.202:80: HTTPConnectionPool(host='50.230.222.202', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2D7A10>, 'Connection to 50.230.222.202 timed out. (connect timeout=5)')))
2024-06-04 13:54:31,274 - ERROR - Proxy error for proxy 50.174.7.156:80: HTTPConnectionPool(host='50.174.7.156', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB27043850>, 'Connection to 50.174.7.156 timed out. (connect timeout=5)')))
2024-06-04 13:54:31,832 - ERROR - Proxy error for proxy 50.217.226.44:80: HTTPConnectionPool(host='50.217.226.44', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2C5D10>, 'Connection to 50.217.226.44 timed out. (connect timeout=5)')))
2024-06-04 13:54:32,033 - ERROR - Proxy error for proxy 50.168.163.176:80: HTTPConnectionPool(host='50.168.163.176', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2E5410>, 'Connection to 50.168.163.176 timed out. (connect timeout=5)')))
2024-06-04 13:54:32,128 - ERROR - Proxy error for proxy 50.168.72.116:80: HTTPConnectionPool(host='50.168.72.116', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB29ECE850>, 'Connection to 50.168.72.116 timed out. (connect timeout=5)')))
2024-06-04 13:54:32,513 - ERROR - Proxy error for proxy 50.168.72.113:80: HTTPConnectionPool(host='50.168.72.113', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2F3890>, 'Connection to 50.168.72.113 timed out. (connect timeout=5)')))
2024-06-04 13:54:32,749 - ERROR - Proxy error for proxy 50.174.7.153:80: HTTPConnectionPool(host='50.174.7.153', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293A3750>, 'Connection to 50.174.7.153 timed out. (connect timeout=5)')))
2024-06-04 13:54:33,183 - ERROR - Proxy error for proxy 50.172.39.98:80: HTTPConnectionPool(host='50.172.39.98', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A3200D0>, 'Connection to 50.172.39.98 timed out. (connect timeout=5)')))
2024-06-04 13:54:33,403 - ERROR - Proxy error for proxy 195.23.57.78:80: HTTPConnectionPool(host='195.23.57.78', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2705E450>, 'Connection to 195.23.57.78 timed out. (connect timeout=5)')))
2024-06-04 13:54:35,532 - ERROR - Proxy error for proxy 50.174.145.13:80: HTTPConnectionPool(host='50.174.145.13', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293F6610>, 'Connection to 50.174.145.13 timed out. (connect timeout=5)')))
2024-06-04 13:54:35,639 - ERROR - Proxy error for proxy 50.222.245.43:80: HTTPConnectionPool(host='50.222.245.43', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB29ECEDD0>, 'Connection to 50.222.245.43 timed out. (connect timeout=5)')))
2024-06-04 13:54:36,291 - ERROR - Proxy error for proxy 50.237.207.186:80: HTTPConnectionPool(host='50.237.207.186', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2940E610>, 'Connection to 50.237.207.186 timed out. (connect timeout=5)')))
2024-06-04 13:54:36,837 - ERROR - Proxy error for proxy 50.207.199.81:80: HTTPConnectionPool(host='50.207.199.81', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293F4C90>, 'Connection to 50.207.199.81 timed out. (connect timeout=5)')))
2024-06-04 13:54:37,038 - ERROR - Proxy error for proxy 50.168.163.180:80: HTTPConnectionPool(host='50.168.163.180', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2D7350>, 'Connection to 50.168.163.180 timed out. (connect timeout=5)')))
2024-06-04 13:54:37,131 - ERROR - Proxy error for proxy 50.223.246.237:80: HTTPConnectionPool(host='50.223.246.237', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293A0E90>, 'Connection to 50.223.246.237 timed out. (connect timeout=5)')))
2024-06-04 13:54:37,520 - ERROR - Proxy error for proxy 50.174.7.162:80: HTTPConnectionPool(host='50.174.7.162', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2DD850>, 'Connection to 50.174.7.162 timed out. (connect timeout=5)')))
2024-06-04 13:54:37,754 - ERROR - Proxy error for proxy 50.222.245.40:80: HTTPConnectionPool(host='50.222.245.40', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB29419890>, 'Connection to 50.222.245.40 timed out. (connect timeout=5)')))
2024-06-04 13:54:38,186 - ERROR - Proxy error for proxy 50.202.75.26:80: HTTPConnectionPool(host='50.202.75.26', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2940F490>, 'Connection to 50.202.75.26 timed out. (connect timeout=5)')))
2024-06-04 13:54:38,683 - ERROR - Proxy error for proxy 8.219.97.248:80: HTTPConnectionPool(host='8.219.97.248', port=80): Read timed out. (read timeout=5)
2024-06-04 13:54:40,548 - ERROR - Proxy error for proxy 50.217.226.40:80: HTTPConnectionPool(host='50.217.226.40', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2D6D10>, 'Connection to 50.217.226.40 timed out. (connect timeout=5)')))
2024-06-04 13:54:40,549 - ERROR - Proxy error for proxy 0.0.0.0:80: HTTPConnectionPool(host='0.0.0.0', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001CB2941A010>: Failed to establish a new connection: [WinError 10049] The requested address is not valid in its context')))
2024-06-04 13:54:40,640 - ERROR - Proxy error for proxy 50.174.145.8:80: HTTPConnectionPool(host='50.174.145.8', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB29ECE250>, 'Connection to 50.174.145.8 timed out. (connect timeout=5)')))
2024-06-04 13:54:41,295 - ERROR - Proxy error for proxy 50.168.72.115:80: HTTPConnectionPool(host='50.168.72.115', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB29E7BC90>, 'Connection to 50.168.72.115 timed out. (connect timeout=5)')))
2024-06-04 13:54:41,843 - ERROR - Proxy error for proxy 50.221.74.130:80: HTTPConnectionPool(host='50.221.74.130', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2D6510>, 'Connection to 50.221.74.130 timed out. (connect timeout=5)')))
2024-06-04 13:54:42,046 - ERROR - Proxy error for proxy 50.231.172.74:80: HTTPConnectionPool(host='50.231.172.74', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293FFE90>, 'Connection to 50.231.172.74 timed out. (connect timeout=5)')))
2024-06-04 13:54:42,139 - ERROR - Proxy error for proxy 50.168.163.166:80: HTTPConnectionPool(host='50.168.163.166', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2940D910>, 'Connection to 50.168.163.166 timed out. (connect timeout=5)')))
2024-06-04 13:54:42,527 - ERROR - Proxy error for proxy 50.171.187.51:80: HTTPConnectionPool(host='50.171.187.51', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A320510>, 'Connection to 50.171.187.51 timed out. (connect timeout=5)')))
2024-06-04 13:54:42,760 - ERROR - Proxy error for proxy 50.168.163.177:80: HTTPConnectionPool(host='50.168.163.177', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2DCF90>, 'Connection to 50.168.163.177 timed out. (connect timeout=5)')))
2024-06-04 13:54:43,196 - ERROR - Proxy error for proxy 50.239.72.19:80: HTTPConnectionPool(host='50.239.72.19', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A32D5D0>, 'Connection to 50.239.72.19 timed out. (connect timeout=5)')))
2024-06-04 13:54:43,694 - ERROR - Proxy error for proxy 50.222.245.46:80: HTTPConnectionPool(host='50.222.245.46', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293A3990>, 'Connection to 50.222.245.46 timed out. (connect timeout=5)')))
2024-06-04 13:54:45,559 - ERROR - Proxy error for proxy 50.172.75.123:80: HTTPConnectionPool(host='50.172.75.123', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293FEC90>, 'Connection to 50.172.75.123 timed out. (connect timeout=5)')))
2024-06-04 13:54:45,652 - ERROR - Proxy error for proxy 50.169.37.50:80: HTTPConnectionPool(host='50.169.37.50', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293F6310>, 'Connection to 50.169.37.50 timed out. (connect timeout=5)')))
2024-06-04 13:54:46,307 - ERROR - Proxy error for proxy 50.174.7.152:80: HTTPConnectionPool(host='50.174.7.152', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB27074850>, 'Connection to 50.174.7.152 timed out. (connect timeout=5)')))
2024-06-04 13:54:46,852 - ERROR - Proxy error for proxy 50.175.212.66:80: HTTPConnectionPool(host='50.175.212.66', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2941A790>, 'Connection to 50.175.212.66 timed out. (connect timeout=5)')))
2024-06-04 13:54:47,053 - ERROR - Proxy error for proxy 50.174.7.154:80: HTTPConnectionPool(host='50.174.7.154', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2D5390>, 'Connection to 50.174.7.154 timed out. (connect timeout=5)')))
2024-06-04 13:54:47,147 - ERROR - Proxy error for proxy 50.174.145.15:80: HTTPConnectionPool(host='50.174.145.15', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2940DC90>, 'Connection to 50.174.145.15 timed out. (connect timeout=5)')))
2024-06-04 13:54:47,535 - ERROR - Proxy error for proxy 32.223.6.94:80: HTTPConnectionPool(host='32.223.6.94', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A301410>, 'Connection to 32.223.6.94 timed out. (connect timeout=5)')))
2024-06-04 13:54:48,204 - ERROR - Proxy error for proxy 50.222.245.45:80: HTTPConnectionPool(host='50.222.245.45', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A322AD0>, 'Connection to 50.222.245.45 timed out. (connect timeout=5)')))
2024-06-04 13:54:48,706 - ERROR - Proxy error for proxy 50.174.7.158:80: HTTPConnectionPool(host='50.174.7.158', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2940C9D0>, 'Connection to 50.174.7.158 timed out. (connect timeout=5)')))
2024-06-04 13:54:49,268 - ERROR - Proxy error for proxy 50.239.72.17:80: HTTPConnectionPool(host='50.239.72.17', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2DCE90>, 'Connection to 50.239.72.17 timed out. (connect timeout=5)')))
2024-06-04 13:54:50,571 - ERROR - Proxy error for proxy 50.223.239.183:80: HTTPConnectionPool(host='50.223.239.183', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293FF710>, 'Connection to 50.223.239.183 timed out. (connect timeout=5)')))
2024-06-04 13:54:50,664 - ERROR - Proxy error for proxy 50.174.7.159:80: HTTPConnectionPool(host='50.174.7.159', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB29ECA510>, 'Connection to 50.174.7.159 timed out. (connect timeout=5)')))
2024-06-04 13:54:51,319 - ERROR - Proxy error for proxy 50.174.145.14:80: HTTPConnectionPool(host='50.174.145.14', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB27131190>, 'Connection to 50.174.145.14 timed out. (connect timeout=5)')))
2024-06-04 13:54:51,864 - ERROR - Proxy error for proxy 50.175.212.76:80: HTTPConnectionPool(host='50.175.212.76', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB27043350>, 'Connection to 50.175.212.76 timed out. (connect timeout=5)')))
2024-06-04 13:54:52,065 - ERROR - Proxy error for proxy 50.171.177.124:80: HTTPConnectionPool(host='50.171.177.124', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2E4310>, 'Connection to 50.171.177.124 timed out. (connect timeout=5)')))
2024-06-04 13:54:52,159 - ERROR - Proxy error for proxy 50.223.239.191:80: HTTPConnectionPool(host='50.223.239.191', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293F7350>, 'Connection to 50.223.239.191 timed out. (connect timeout=5)')))
2024-06-04 13:54:52,546 - ERROR - Proxy error for proxy 50.174.7.155:80: HTTPConnectionPool(host='50.174.7.155', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2F32D0>, 'Connection to 50.174.7.155 timed out. (connect timeout=5)')))
2024-06-04 13:54:53,219 - ERROR - Proxy error for proxy 50.145.24.181:80: HTTPConnectionPool(host='50.145.24.181', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2C7D90>, 'Connection to 50.145.24.181 timed out. (connect timeout=5)')))
2024-06-04 13:54:53,714 - ERROR - Proxy error for proxy 50.239.72.18:80: HTTPConnectionPool(host='50.239.72.18', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A3016D0>, 'Connection to 50.239.72.18 timed out. (connect timeout=5)')))
2024-06-04 13:54:54,276 - ERROR - Proxy error for proxy 50.223.239.167:80: HTTPConnectionPool(host='50.223.239.167', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2E65D0>, 'Connection to 50.223.239.167 timed out. (connect timeout=5)')))
2024-06-04 13:54:55,674 - ERROR - Proxy error for proxy 68.185.57.66:80: HTTPConnectionPool(host='68.185.57.66', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2D6610>, 'Connection to 68.185.57.66 timed out. (connect timeout=5)')))
2024-06-04 13:54:55,783 - ERROR - Proxy error for proxy 50.223.239.166:80: HTTPConnectionPool(host='50.223.239.166', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB29ECF4D0>, 'Connection to 50.223.239.166 timed out. (connect timeout=5)')))
2024-06-04 13:54:56,325 - ERROR - Proxy error for proxy 156.239.50.176:3128: HTTPConnectionPool(host='156.239.50.176', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2705CA90>, 'Connection to 156.239.50.176 timed out. (connect timeout=5)')))
2024-06-04 13:54:57,256 - ERROR - Proxy error for proxy 156.239.55.151:3128: HTTPConnectionPool(host='156.239.55.151', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A337910>, 'Connection to 156.239.55.151 timed out. (connect timeout=5)')))
2024-06-04 13:54:59,284 - ERROR - Proxy error for proxy 156.239.55.49:3128: HTTPConnectionPool(host='156.239.55.49', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2C6A50>, 'Connection to 156.239.55.49 timed out. (connect timeout=5)')))
2024-06-04 13:54:59,813 - ERROR - Proxy error for proxy 104.165.169.150:3128: HTTPConnectionPool(host='104.165.169.150', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A349C50>, 'Connection to 104.165.169.150 timed out. (connect timeout=5)')))
2024-06-04 13:55:01,684 - ERROR - Proxy error for proxy 156.239.48.96:3128: HTTPConnectionPool(host='156.239.48.96', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293F6DD0>, 'Connection to 156.239.48.96 timed out. (connect timeout=5)')))
2024-06-04 13:55:02,057 - ERROR - Proxy error for proxy 104.165.127.165:3128: HTTPConnectionPool(host='104.165.127.165', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293F4C90>, 'Connection to 104.165.127.165 timed out. (connect timeout=5)')))
2024-06-04 13:55:02,477 - ERROR - Proxy error for proxy 104.164.183.176:3128: HTTPConnectionPool(host='104.164.183.176', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2F0090>, 'Connection to 104.164.183.176 timed out. (connect timeout=5)')))
2024-06-04 13:55:03,080 - ERROR - Proxy error for proxy 156.239.55.41:3128: HTTPConnectionPool(host='156.239.55.41', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2941D110>, 'Connection to 156.239.55.41 timed out. (connect timeout=5)')))
2024-06-04 13:55:03,610 - ERROR - Proxy error for proxy 104.165.169.240:3128: HTTPConnectionPool(host='104.165.169.240', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2C4E90>, 'Connection to 104.165.169.240 timed out. (connect timeout=5)')))
2024-06-04 13:55:03,954 - ERROR - Proxy error for proxy 104.252.131.217:3128: HTTPConnectionPool(host='104.252.131.217', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB29ECEDD0>, 'Connection to 104.252.131.217 timed out. (connect timeout=5)')))
2024-06-04 13:55:04,293 - ERROR - Proxy error for proxy 104.165.127.40:3128: HTTPConnectionPool(host='104.165.127.40', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2DF650>, 'Connection to 104.165.127.40 timed out. (connect timeout=5)')))
2024-06-04 13:55:07,452 - ERROR - Proxy error for proxy 144.24.204.126:8118: ("Connection broken: ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None)", ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None))
2024-06-04 13:55:07,481 - ERROR - Proxy error for proxy 156.239.53.22:3128: HTTPConnectionPool(host='156.239.53.22', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2C6ED0>, 'Connection to 156.239.53.22 timed out. (connect timeout=5)')))
2024-06-04 13:55:08,096 - ERROR - Proxy error for proxy 156.239.49.144:3128: HTTPConnectionPool(host='156.239.49.144', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A350310>, 'Connection to 156.239.49.144 timed out. (connect timeout=5)')))
2024-06-04 13:55:08,358 - ERROR - Proxy error for proxy 156.239.48.128:3128: HTTPConnectionPool(host='156.239.48.128', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB26F3A690>, 'Connection to 156.239.48.128 timed out. (connect timeout=5)')))
2024-06-04 13:55:10,317 - ERROR - Proxy error for proxy 201.77.108.21:999: HTTPConnectionPool(host='201.77.108.21', port=999): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2D6350>, 'Connection to 201.77.108.21 timed out. (connect timeout=5)')))
2024-06-04 13:55:10,504 - ERROR - Proxy error for proxy 194.163.181.18:3128: HTTPConnectionPool(host='194.163.181.18', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB29ECF690>, 'Connection to 194.163.181.18 timed out. (connect timeout=5)')))
2024-06-04 13:55:10,767 - ERROR - Proxy error for proxy 104.164.183.246:3128: HTTPConnectionPool(host='104.164.183.246', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2C6490>, 'Connection to 104.164.183.246 timed out. (connect timeout=5)')))
2024-06-04 13:55:13,329 - ERROR - Proxy error for proxy 104.165.169.180:3128: HTTPConnectionPool(host='104.165.169.180', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2D7490>, 'Connection to 104.165.169.180 timed out. (connect timeout=5)')))
2024-06-04 13:55:14,463 - ERROR - Proxy error for proxy 104.164.183.71:3128: HTTPConnectionPool(host='104.164.183.71', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A300F50>, 'Connection to 104.164.183.71 timed out. (connect timeout=5)')))
2024-06-04 13:55:15,332 - ERROR - Proxy error for proxy 104.252.131.170:3128: HTTPConnectionPool(host='104.252.131.170', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2C7890>, 'Connection to 104.252.131.170 timed out. (connect timeout=5)')))
2024-06-04 13:55:15,705 - ERROR - Proxy error for proxy 156.239.48.10:3128: HTTPConnectionPool(host='156.239.48.10', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293F6ED0>, 'Connection to 156.239.48.10 timed out. (connect timeout=5)')))
2024-06-04 13:55:15,782 - ERROR - Proxy error for proxy 104.164.183.129:3128: HTTPConnectionPool(host='104.164.183.129', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A350F50>, 'Connection to 104.164.183.129 timed out. (connect timeout=5)')))
2024-06-04 13:55:18,929 - ERROR - Proxy error for proxy 156.239.49.138:3128: HTTPConnectionPool(host='156.239.49.138', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293F4190>, 'Connection to 156.239.49.138 timed out. (connect timeout=5)')))
2024-06-04 13:55:20,604 - ERROR - Proxy error for proxy 156.239.48.180:3128: HTTPConnectionPool(host='156.239.48.180', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2D7E50>, 'Connection to 156.239.48.180 timed out. (connect timeout=5)')))
2024-06-04 13:55:21,631 - ERROR - Proxy error for proxy 156.239.52.44:3128: HTTPConnectionPool(host='156.239.52.44', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB29418810>, 'Connection to 156.239.52.44 timed out. (connect timeout=5)')))
2024-06-04 13:55:22,749 - ERROR - Proxy error for proxy 104.252.131.10:3128: HTTPConnectionPool(host='104.252.131.10', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2E5850>, 'Connection to 104.252.131.10 timed out. (connect timeout=5)')))
2024-06-04 13:55:23,093 - ERROR - Proxy error for proxy 104.165.127.101:3128: HTTPConnectionPool(host='104.165.127.101', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB27074850>, 'Connection to 104.165.127.101 timed out. (connect timeout=5)')))
2024-06-04 13:55:24,660 - ERROR - Proxy error for proxy 156.239.50.148:3128: HTTPConnectionPool(host='156.239.50.148', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2705F4D0>, 'Connection to 156.239.50.148 timed out. (connect timeout=5)')))
2024-06-04 13:55:25,301 - ERROR - Proxy error for proxy 156.239.49.84:3128: HTTPConnectionPool(host='156.239.49.84', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2941CC10>, 'Connection to 156.239.49.84 timed out. (connect timeout=5)')))
2024-06-04 13:55:26,243 - ERROR - Proxy error for proxy 104.164.183.215:3128: HTTPConnectionPool(host='104.164.183.215', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A336BD0>, 'Connection to 104.164.183.215 timed out. (connect timeout=5)')))
2024-06-04 13:55:26,528 - ERROR - Proxy error for proxy 67.43.227.228:12643: HTTPConnectionPool(host='67.43.227.228', port=12643): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001CB29ECE9D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')))
2024-06-04 13:55:27,878 - ERROR - Proxy error for proxy 183.100.14.134:8000: HTTPConnectionPool(host='183.100.14.134', port=8000): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB29ECF110>, 'Connection to 183.100.14.134 timed out. (connect timeout=5)')))
2024-06-04 13:55:28,098 - ERROR - Proxy error for proxy 201.218.200.115:8080: HTTPConnectionPool(host='201.218.200.115', port=8080): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2C7110>, 'Connection to 201.218.200.115 timed out. (connect timeout=5)')))
2024-06-04 13:55:28,720 - ERROR - Proxy error for proxy 85.209.154.148:33333: HTTPConnectionPool(host='85.209.154.148', port=33333): Read timed out. (read timeout=5)
2024-06-04 13:55:28,938 - ERROR - Proxy error for proxy 4.155.2.13:9480: HTTPConnectionPool(host='4.155.2.13', port=9480): Read timed out. (read timeout=5)
2024-06-04 13:55:28,954 - ERROR - Proxy error for proxy 117.54.114.96:80: HTTPConnectionPool(host='117.54.114.96', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB27043010>, 'Connection to 117.54.114.96 timed out. (connect timeout=5)')))
2024-06-04 13:55:29,665 - ERROR - Proxy error for proxy 217.66.25.133:8060: HTTPConnectionPool(host='217.66.25.133', port=8060): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB292C8AD0>, 'Connection to 217.66.25.133 timed out. (connect timeout=5)')))
2024-06-04 13:55:30,397 - ERROR - Proxy error for proxy 47.251.43.115:33333: HTTPConnectionPool(host='47.251.43.115', port=33333): Read timed out. (read timeout=5)
2024-06-04 13:55:30,815 - ERROR - Proxy error for proxy 162.223.94.164:80: HTTPConnectionPool(host='162.223.94.164', port=80): Read timed out. (read timeout=5)
2024-06-04 13:55:31,100 - ERROR - Proxy error for proxy 72.10.160.92:27103: HTTPConnectionPool(host='72.10.160.92', port=27103): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001CB2705C290>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')))
2024-06-04 13:55:34,661 - ERROR - Proxy error for proxy 200.33.112.55:80: HTTPConnectionPool(host='200.33.112.55', port=80): Read timed out. (read timeout=5)
2024-06-04 13:55:35,175 - ERROR - Proxy error for proxy 50.223.246.226:80: HTTPConnectionPool(host='50.223.246.226', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A35FE90>, 'Connection to 50.223.246.226 timed out. (connect timeout=5)')))
2024-06-04 13:55:35,191 - ERROR - Proxy error for proxy 50.171.68.130:80: HTTPConnectionPool(host='50.171.68.130', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A322E50>, 'Connection to 50.171.68.130 timed out. (connect timeout=5)')))
2024-06-04 13:55:35,220 - ERROR - Proxy error for proxy 50.218.204.103:80: HTTPConnectionPool(host='50.218.204.103', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293F7810>, 'Connection to 50.218.204.103 timed out. (connect timeout=5)')))
2024-06-04 13:55:35,407 - ERROR - Proxy error for proxy 50.223.239.168:80: HTTPConnectionPool(host='50.223.239.168', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A367450>, 'Connection to 50.223.239.168 timed out. (connect timeout=5)')))
2024-06-04 13:55:35,592 - ERROR - Proxy error for proxy 50.168.72.122:80: HTTPConnectionPool(host='50.168.72.122', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2941CBD0>, 'Connection to 50.168.72.122 timed out. (connect timeout=5)')))
2024-06-04 13:55:35,794 - ERROR - Proxy error for proxy 50.144.161.167:80: HTTPConnectionPool(host='50.144.161.167', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2D59D0>, 'Connection to 50.144.161.167 timed out. (connect timeout=5)')))
2024-06-04 13:55:35,827 - ERROR - Proxy error for proxy 50.175.212.79:80: HTTPConnectionPool(host='50.175.212.79', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2C4590>, 'Connection to 50.175.212.79 timed out. (connect timeout=5)')))
2024-06-04 13:55:36,106 - ERROR - Proxy error for proxy 213.143.113.82:80: HTTPConnectionPool(host='213.143.113.82', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2941B3D0>, 'Connection to 213.143.113.82 timed out. (connect timeout=5)')))
2024-06-04 13:55:38,660 - ERROR - Proxy error for proxy 58.176.49.86:80: HTTPConnectionPool(host='58.176.49.86', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001CB293F4810>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')))
2024-06-04 13:55:39,573 - ERROR - Proxy error for proxy 50.207.199.86:80: HTTPConnectionPool(host='50.207.199.86', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2D6C90>, 'Connection to 50.207.199.86 timed out. (connect timeout=5)')))
2024-06-04 13:55:39,667 - ERROR - Proxy error for proxy 50.172.75.120:80: HTTPConnectionPool(host='50.172.75.120', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2F3190>, 'Connection to 50.172.75.120 timed out. (connect timeout=5)')))
2024-06-04 13:55:40,182 - ERROR - Proxy error for proxy 50.171.122.30:80: HTTPConnectionPool(host='50.171.122.30', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB29ECE910>, 'Connection to 50.171.122.30 timed out. (connect timeout=5)')))
2024-06-04 13:55:40,228 - ERROR - Proxy error for proxy 90.84.17.133:3128: HTTPConnectionPool(host='90.84.17.133', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293EB050>, 'Connection to 90.84.17.133 timed out. (connect timeout=5)')))
2024-06-04 13:55:40,510 - ERROR - Proxy error for proxy 38.56.70.75:999: HTTPConnectionPool(host='38.56.70.75', port=999): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB29ECE410>, 'Connection to 38.56.70.75 timed out. (connect timeout=5)')))
2024-06-04 13:55:40,602 - ERROR - Proxy error for proxy 152.32.64.43:8095: HTTPConnectionPool(host='152.32.64.43', port=8095): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293F41D0>, 'Connection to 152.32.64.43 timed out. (connect timeout=5)')))
2024-06-04 13:55:40,805 - ERROR - Proxy error for proxy 113.23.183.154:8989: HTTPConnectionPool(host='113.23.183.154', port=8989): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2705D390>, 'Connection to 113.23.183.154 timed out. (connect timeout=5)')))
2024-06-04 13:55:40,836 - ERROR - Proxy error for proxy 103.170.100.96:8080: HTTPConnectionPool(host='103.170.100.96', port=8080): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2F0210>, 'Connection to 103.170.100.96 timed out. (connect timeout=5)')))
2024-06-04 13:55:41,118 - ERROR - Proxy error for proxy 68.183.14.206:3128: HTTPConnectionPool(host='68.183.14.206', port=3128): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB27074E50>, 'Connection to 68.183.14.206 timed out. (connect timeout=5)')))
2024-06-04 13:55:41,680 - ERROR - Proxy error for proxy 117.250.3.58:8080: ('Connection broken: IncompleteRead(3254 bytes read, 5909 more expected)', IncompleteRead(3254 bytes read, 5909 more expected))
2024-06-04 13:55:43,661 - ERROR - Proxy error for proxy 217.100.239.117:8090: HTTPConnectionPool(host='217.100.239.117', port=8090): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A367ED0>, 'Connection to 217.100.239.117 timed out. (connect timeout=5)')))
2024-06-04 13:55:44,672 - ERROR - Proxy error for proxy 66.191.31.158:80: HTTPConnectionPool(host='66.191.31.158', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB293F6C10>, 'Connection to 66.191.31.158 timed out. (connect timeout=5)')))
2024-06-04 13:55:45,184 - ERROR - Proxy error for proxy 213.33.126.130:80: HTTPConnectionPool(host='213.33.126.130', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A323A90>, 'Connection to 213.33.126.130 timed out. (connect timeout=5)')))
2024-06-04 13:55:45,231 - ERROR - Proxy error for proxy 50.144.161.162:80: HTTPConnectionPool(host='50.144.161.162', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2E6750>, 'Connection to 50.144.161.162 timed out. (connect timeout=5)')))
2024-06-04 13:55:45,525 - ERROR - Proxy error for proxy 50.168.72.118:80: HTTPConnectionPool(host='50.168.72.118', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB292C8B50>, 'Connection to 50.168.72.118 timed out. (connect timeout=5)')))
2024-06-04 13:55:45,617 - ERROR - Proxy error for proxy 213.157.6.50:80: HTTPConnectionPool(host='213.157.6.50', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2F0A50>, 'Connection to 213.157.6.50 timed out. (connect timeout=5)')))
2024-06-04 13:55:46,132 - ERROR - Proxy error for proxy 50.168.72.119:80: HTTPConnectionPool(host='50.168.72.119', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A35D210>, 'Connection to 50.168.72.119 timed out. (connect timeout=5)')))
2024-06-04 13:55:46,690 - ERROR - Proxy error for proxy 50.175.212.72:80: HTTPConnectionPool(host='50.175.212.72', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2705F990>, 'Connection to 50.175.212.72 timed out. (connect timeout=5)')))
2024-06-04 13:55:46,690 - ERROR - Proxy error for proxy 50.217.226.45:80: HTTPConnectionPool(host='50.217.226.45', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2705ED50>, 'Connection to 50.217.226.45 timed out. (connect timeout=5)')))
2024-06-04 13:55:47,733 - ERROR - Proxy error for proxy 62.37.124.180:3128: HTTPConnectionPool(host='62.37.124.180', port=3128): Read timed out. (read timeout=5)
2024-06-04 13:55:47,810 - ERROR - Proxy error for proxy 127.0.0.7:80: HTTPConnectionPool(host='127.0.0.7', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001CB2A35DC90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')))
2024-06-04 13:55:48,670 - ERROR - Proxy error for proxy 50.207.199.83:80: HTTPConnectionPool(host='50.207.199.83', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2F3850>, 'Connection to 50.207.199.83 timed out. (connect timeout=5)')))
2024-06-04 13:55:48,832 - ERROR - Proxy error for proxy 72.10.160.91:17137: HTTPConnectionPool(host='72.10.160.91', port=17137): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001CB2941CDD0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')))
2024-06-04 13:55:49,681 - ERROR - Proxy error for proxy 50.207.199.87:80: HTTPConnectionPool(host='50.207.199.87', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2941E3D0>, 'Connection to 50.207.199.87 timed out. (connect timeout=5)')))
2024-06-04 13:55:49,681 - ERROR - Proxy error for proxy 2024-06-04:312: HTTPConnectionPool(host='2024-06-04', port=312): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NameResolutionError("<urllib3.connection.HTTPConnection object at 0x000001CB2A2BE2D0>: Failed to resolve '2024-06-04' ([Errno 11001] getaddrinfo failed)")))
2024-06-04 13:55:50,239 - ERROR - Proxy error for proxy 50.172.75.124:80: HTTPConnectionPool(host='50.172.75.124', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2E5090>, 'Connection to 50.172.75.124 timed out. (connect timeout=5)')))
2024-06-04 13:55:50,459 - ERROR - Proxy error for proxy 2024-06-03:352: HTTPConnectionPool(host='2024-06-03', port=352): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NameResolutionError("<urllib3.connection.HTTPConnection object at 0x000001CB293A3CD0>: Failed to resolve '2024-06-03' ([Errno 11001] getaddrinfo failed)")))
2024-06-04 13:55:50,537 - ERROR - Proxy error for proxy 85.8.68.2:80: HTTPConnectionPool(host='85.8.68.2', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB29ECF950>, 'Connection to 85.8.68.2 timed out. (connect timeout=5)')))
2024-06-04 13:55:50,537 - ERROR - Proxy error for proxy 2024-06-02:476: HTTPConnectionPool(host='2024-06-02', port=476): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NameResolutionError("<urllib3.connection.HTTPConnection object at 0x000001CB27042ED0>: Failed to resolve '2024-06-02' ([Errno 11001] getaddrinfo failed)")))
2024-06-04 13:55:50,630 - ERROR - Proxy error for proxy 50.168.7.250:80: HTTPConnectionPool(host='50.168.7.250', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2A2E5610>, 'Connection to 50.168.7.250 timed out. (connect timeout=5)')))
2024-06-04 13:55:51,143 - ERROR - Proxy error for proxy 50.168.163.179:80: HTTPConnectionPool(host='50.168.163.179', port=80): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001CB2941B1D0>, 'Connection to 50.168.163.179 timed out. (connect timeout=5)')))
2024-06-04 13:55:51,407 - ERROR - Proxy error for proxy 2024-06-01:402: HTTPConnectionPool(host='2024-06-01', port=402): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NameResolutionError("<urllib3.connection.HTTPConnection object at 0x000001CB293F6890>: Failed to resolve '2024-06-01' ([Errno 11001] getaddrinfo failed)")))
2024-06-04 13:55:51,545 - ERROR - Proxy error for proxy 2024-05-31:367: HTTPConnectionPool(host='2024-05-31', port=367): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NameResolutionError("<urllib3.connection.HTTPConnection object at 0x000001CB2941A590>: Failed to resolve '2024-05-31' ([Errno 11001] getaddrinfo failed)")))
2024-06-04 13:55:52,399 - ERROR - Proxy error for proxy 380:91: HTTPConnectionPool(host='380', port=91): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NameResolutionError("<urllib3.connection.HTTPConnection object at 0x000001CB2A35D750>: Failed to resolve '380' ([Errno 11001] getaddrinfo failed)")))
2024-06-04 13:55:52,416 - ERROR - Proxy error for proxy 175:42: HTTPConnectionPool(host='175', port=42): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NameResolutionError("<urllib3.connection.HTTPConnection object at 0x000001CB29ECF3D0>: Failed to resolve '175' ([Errno 11001] getaddrinfo failed)")))
2024-06-04 13:55:52,948 - ERROR - Proxy error for proxy 58:14: HTTPConnectionPool(host='58', port=14): Max retries exceeded with url: http://www.google.com/ (Caused by ProxyError('Unable to connect to proxy', NameResolutionError("<urllib3.connection.HTTPConnection object at 0x000001CB2A2E5D10>: Failed to resolve '58' ([Errno 11001] getaddrinfo failed)")))
2024-06-04 13:55:53,355 - ERROR - Request failed (retries left: 4): HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /search?q=This+work+was+supported+in+part+through+the+NYU+IT+High+Performance+Computing+resources%2C+services%2C+and+staff+expertise&start=10 (Caused by ProxyError('Unable to connect to proxy. Your proxy appears to only use HTTP and not HTTPS, try changing your proxy URL to be HTTP. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#https-proxy-error-http-proxy', SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1006)'))))
2024-06-04 13:55:53,355 - INFO - Removed invalid proxy: 54.234.212.205:3128
2024-06-04 13:55:53,495 - ERROR - Request failed (retries left: 4): HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /search?q=This+work+was+supported+in+part+through+the+NYU+IT+High+Performance+Computing+resources%2C+services%2C+and+staff+expertise&start=20 (Caused by ProxyError('Unable to connect to proxy', SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1006)'))))
2024-06-04 13:55:53,495 - INFO - Removed invalid proxy: 13.56.163.250:3128
2024-06-04 13:55:53,639 - ERROR - Request failed (retries left: 4): HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /search?q=This+work+was+supported+in+part+through+the+NYU+IT+High+Performance+Computing+resources%2C+services%2C+and+staff+expertise&start=30 (Caused by ProxyError('Unable to connect to proxy. Your proxy appears to only use HTTP and not HTTPS, try changing your proxy URL to be HTTP. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#https-proxy-error-http-proxy', SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1006)'))))
2024-06-04 13:55:53,639 - INFO - Removed invalid proxy: 161.35.70.249:3128
2024-06-04 13:55:54,049 - ERROR - Request failed (retries left: 4): HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /search?q=This+work+was+supported+in+part+through+the+NYU+IT+High+Performance+Computing+resources%2C+services%2C+and+staff+expertise&start=40 (Caused by ProxyError('Unable to connect to proxy. Your proxy appears to only use HTTP and not HTTPS, try changing your proxy URL to be HTTP. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#https-proxy-error-http-proxy', SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1006)'))))
2024-06-04 13:55:54,049 - INFO - Removed invalid proxy: 213.176.6.101:80
2024-06-04 13:55:56,374 - ERROR - Request failed (retries left: 4): HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /search?q=This+work+was+supported+in+part+through+the+NYU+IT+High+Performance+Computing+resources%2C+services%2C+and+staff+expertise&start=0 (Caused by ProxyError('Unable to connect to proxy', NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001CB2A321990>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')))
2024-06-04 13:55:56,375 - INFO - Removed invalid proxy: 122.155.165.191:3128
2024-06-04 13:56:02,979 - ERROR - Request failed (retries left: 3): HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /search?q=This+work+was+supported+in+part+through+the+NYU+IT+High+Performance+Computing+resources%2C+services%2C+and+staff+expertise&start=40 (Caused by ProxyError('Unable to connect to proxy. Your proxy appears to only use HTTP and not HTTPS, try changing your proxy URL to be HTTP. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#https-proxy-error-http-proxy', SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1006)'))))
2024-06-04 13:56:02,979 - INFO - Removed invalid proxy: 188.132.209.245:80
2024-06-04 13:56:03,336 - ERROR - Request failed (retries left: 3): HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /search?q=This+work+was+supported+in+part+through+the+NYU+IT+High+Performance+Computing+resources%2C+services%2C+and+staff+expertise&start=20 (Caused by ProxyError('Unable to connect to proxy', SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1006)'))))
2024-06-04 13:56:03,337 - INFO - Removed invalid proxy: 13.56.163.250:3128
2024-06-04 13:56:06,593 - ERROR - Request failed (retries left: 3): HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /search?q=This+work+was+supported+in+part+through+the+NYU+IT+High+Performance+Computing+resources%2C+services%2C+and+staff+expertise&start=30 (Caused by ProxyError('Unable to connect to proxy. Your proxy appears to only use HTTP and not HTTPS, try changing your proxy URL to be HTTP. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#https-proxy-error-http-proxy', SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1006)'))))
2024-06-04 13:56:06,593 - INFO - Removed invalid proxy: 51.89.73.162:80
2024-06-04 13:56:07,958 - ERROR - Request failed (retries left: 3): HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /search?q=This+work+was+supported+in+part+through+the+NYU+IT+High+Performance+Computing+resources%2C+services%2C+and+staff+expertise&start=0 (Caused by ProxyError('Unable to connect to proxy. Your proxy appears to only use HTTP and not HTTPS, try changing your proxy URL to be HTTP. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#https-proxy-error-http-proxy', SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1006)'))))
2024-06-04 13:56:07,958 - INFO - Removed invalid proxy: 51.210.19.141:80
2024-06-04 13:56:08,990 - ERROR - Request failed (retries left: 3): HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /search?q=This+work+was+supported+in+part+through+the+NYU+IT+High+Performance+Computing+resources%2C+services%2C+and+staff+expertise&start=10 (Caused by ProxyError('Unable to connect to proxy', ReadTimeoutError("HTTPSConnectionPool(host='www.google.com', port=443): Read timed out. (read timeout=10)")))
2024-06-04 13:56:08,990 - INFO - Removed invalid proxy: 182.253.109.59:8080
2024-06-04 13:56:09,525 - ERROR - Request failed (retries left: 2): HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /search?q=This+work+was+supported+in+part+through+the+NYU+IT+High+Performance+Computing+resources%2C+services%2C+and+staff+expertise&start=20 (Caused by ProxyError('Unable to connect to proxy. Your proxy appears to only use HTTP and not HTTPS, try changing your proxy URL to be HTTP. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#https-proxy-error-http-proxy', SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1006)'))))
2024-06-04 13:56:09,525 - INFO - Removed invalid proxy: 35.185.196.38:3128
2024-06-04 13:56:12,202 - ERROR - Request failed (retries left: 2): HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /search?q=This+work+was+supported+in+part+through+the+NYU+IT+High+Performance+Computing+resources%2C+services%2C+and+staff+expertise&start=40 (Caused by ProxyError('Unable to connect to proxy. Your proxy appears to only use HTTP and not HTTPS, try changing your proxy URL to be HTTP. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#https-proxy-error-http-proxy', SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1006)'))))
2024-06-04 13:56:12,202 - INFO - Removed invalid proxy: 38.54.71.67:80
2024-06-04 13:56:12,475 - ERROR - Request failed (retries left: 2): HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /search?q=This+work+was+supported+in+part+through+the+NYU+IT+High+Performance+Computing+resources%2C+services%2C+and+staff+expertise&start=30 (Caused by ProxyError('Unable to connect to proxy. Your proxy appears to only use HTTP and not HTTPS, try changing your proxy URL to be HTTP. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#https-proxy-error-http-proxy', SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1006)'))))
2024-06-04 13:56:12,475 - INFO - Removed invalid proxy: 37.27.29.181:80
2024-06-04 13:56:15,499 - ERROR - Request failed (retries left: 2): HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /search?q=This+work+was+supported+in+part+through+the+NYU+IT+High+Performance+Computing+resources%2C+services%2C+and+staff+expertise&start=0 (Caused by ProxyError('Unable to connect to proxy. Your proxy appears to only use HTTP and not HTTPS, try changing your proxy URL to be HTTP. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#https-proxy-error-http-proxy', SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1006)'))))
2024-06-04 13:56:15,499 - INFO - Removed invalid proxy: 137.184.121.54:80
2024-06-04 13:56:24,520 - ERROR - Request failed (retries left: 1): HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /search?q=This+work+was+supported+in+part+through+the+NYU+IT+High+Performance+Computing+resources%2C+services%2C+and+staff+expertise&start=40 (Caused by ProxyError('Unable to connect to proxy', SSLError(SSLEOFError(8, '[SSL: UNEXPECTED_EOF_WHILE_READING] EOF occurred in violation of protocol (_ssl.c:1006)'))))
2024-06-04 13:56:24,520 - INFO - Removed invalid proxy: 147.75.88.36:10001
2024-06-04 13:56:25,460 - ERROR - Request failed (retries left: 1): HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /search?q=This+work+was+supported+in+part+through+the+NYU+IT+High+Performance+Computing+resources%2C+services%2C+and+staff+expertise&start=30 (Caused by ProxyError('Unable to connect to proxy. Your proxy appears to only use HTTP and not HTTPS, try changing your proxy URL to be HTTP. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#https-proxy-error-http-proxy', SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1006)'))))
2024-06-04 13:56:25,460 - INFO - Removed invalid proxy: 185.159.153.109:80
2024-06-04 13:56:28,175 - ERROR - Request failed (retries left: 1): HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /search?q=This+work+was+supported+in+part+through+the+NYU+IT+High+Performance+Computing+resources%2C+services%2C+and+staff+expertise&start=20 (Caused by ProxyError('Unable to connect to proxy', ReadTimeoutError("HTTPSConnectionPool(host='www.google.com', port=443): Read timed out. (read timeout=10)")))
2024-06-04 13:56:28,175 - INFO - Removed invalid proxy: 37.27.82.72:80
2024-06-04 13:56:28,261 - ERROR - Request failed (retries left: 2): HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /search?q=This+work+was+supported+in+part+through+the+NYU+IT+High+Performance+Computing+resources%2C+services%2C+and+staff+expertise&start=10 (Caused by ProxyError('Unable to connect to proxy. Your proxy appears to only use HTTP and not HTTPS, try changing your proxy URL to be HTTP. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#https-proxy-error-http-proxy', SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1006)'))))
2024-06-04 13:56:28,261 - INFO - Removed invalid proxy: 216.137.184.253:80
2024-06-04 13:56:31,243 - ERROR - Request failed (retries left: 1): HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /search?q=This+work+was+supported+in+part+through+the+NYU+IT+High+Performance+Computing+resources%2C+services%2C+and+staff+expertise&start=0 (Caused by ProxyError('Unable to connect to proxy. Your proxy appears to only use HTTP and not HTTPS, try changing your proxy URL to be HTTP. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#https-proxy-error-http-proxy', SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1006)'))))
2024-06-04 13:56:31,243 - INFO - Removed invalid proxy: 139.59.1.14:8080
2024-06-04 13:56:38,979 - ERROR - Request failed (retries left: 0): HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /search?q=This+work+was+supported+in+part+through+the+NYU+IT+High+Performance+Computing+resources%2C+services%2C+and+staff+expertise&start=40 (Caused by ProxyError('Unable to connect to proxy. Your proxy appears to only use HTTP and not HTTPS, try changing your proxy URL to be HTTP. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#https-proxy-error-http-proxy', SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1006)'))))
2024-06-04 13:56:38,979 - INFO - Removed invalid proxy: 172.96.117.205:38001
2024-06-04 13:56:42,747 - ERROR - Request failed (retries left: 1): HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /search?q=This+work+was+supported+in+part+through+the+NYU+IT+High+Performance+Computing+resources%2C+services%2C+and+staff+expertise&start=10 (Caused by ProxyError('Unable to connect to proxy. Your proxy appears to only use HTTP and not HTTPS, try changing your proxy URL to be HTTP. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#https-proxy-error-http-proxy', SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1006)'))))
2024-06-04 13:56:42,747 - INFO - Removed invalid proxy: 65.109.189.49:80
2024-06-04 13:56:45,560 - ERROR - Request failed (retries left: 0): HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /search?q=This+work+was+supported+in+part+through+the+NYU+IT+High+Performance+Computing+resources%2C+services%2C+and+staff+expertise&start=0 (Caused by ProxyError('Unable to connect to proxy. Your proxy appears to only use HTTP and not HTTPS, try changing your proxy URL to be HTTP. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#https-proxy-error-http-proxy', SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1006)'))))
2024-06-04 13:56:45,560 - INFO - Removed invalid proxy: 210.212.39.138:8080
2024-06-04 13:56:45,671 - INFO - Fetched HTML content for page 4 without proxy
2024-06-04 13:56:45,703 - INFO - Fetched results: [{'Title': 'Google Developer Student Clubs', 'Link': 'https://developers.google.com/community/gdsc'}, {'Title': 'Learning Assessment Tools & Software', 'Link': 'https://examsoft.com/'}, {'Title': 'Key Projects and Initiatives', 'Link': 'https://www.nyu.edu/life/information-technology/about-nyu-it/key-projects-and-initiatives.html'}, {'Title': 'NYU Langone Fertility Center', 'Link': 'https://www.fertilityiq.com/fertilityiq/clinics/nyu-langone-fertility-center'}, {'Title': 'What Are Soft Skills? Definition, Importance, and Examples', 'Link': 'https://www.investopedia.com/terms/s/soft-skills.asp'}, {'Title': 'Center for Research Computing - NYU Abu Dhabi', 'Link': 'https://nyuad.nyu.edu/en/about/leadership-and-administration/office-of-the-provost/research/research-and-research-institute/center-for-research-computing.html'}, {'Title': 'Shanif Dhanani - Locusive', 'Link': 'https://www.linkedin.com/in/shanifdhanani'}, {'Title': 'Better, Faster, Greener - NYU Launches One of The Fastest ...', 'Link': 'https://www.nyu.edu/about/news-publications/news/2020/november/GreeneSupercomputer.html'}, {'Title': 'Research Electronic Data Capture (REDCap)', 'Link': 'https://www.nyu.edu/life/information-technology/research-computing-services/research-data-and-tools/redcap.html'}, {'Title': 'Research - NYU Abu Dhabi', 'Link': 'https://nyuad.nyu.edu/en/research.html'}]
2024-06-04 13:56:46,781 - ERROR - Request failed (retries left: 0): HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /search?q=This+work+was+supported+in+part+through+the+NYU+IT+High+Performance+Computing+resources%2C+services%2C+and+staff+expertise&start=30 (Caused by ProxyError('Unable to connect to proxy', ReadTimeoutError("HTTPSConnectionPool(host='www.google.com', port=443): Read timed out. (read timeout=10)")))
2024-06-04 13:56:46,781 - INFO - Removed invalid proxy: 209.146.104.56:80
2024-06-04 13:56:50,604 - ERROR - Request failed (retries left: 0): HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /search?q=This+work+was+supported+in+part+through+the+NYU+IT+High+Performance+Computing+resources%2C+services%2C+and+staff+expertise&start=20 (Caused by ProxyError('Unable to connect to proxy', ReadTimeoutError("HTTPSConnectionPool(host='www.google.com', port=443): Read timed out. (read timeout=10)")))
2024-06-04 13:56:50,604 - INFO - Removed invalid proxy: 116.203.28.43:80
2024-06-04 13:56:54,734 - INFO - Fetched HTML content for page 3 without proxy
2024-06-04 13:56:54,749 - WARNING - No search results found on the page.
2024-06-04 13:56:54,751 - INFO - Fetched results: []
2024-06-04 13:56:55,291 - ERROR - Request failed (retries left: 4): HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /search?q=This+work+was+supported+in+part+through+the+NYU+IT+High+Performance+Computing+resources%2C+services%2C+and+staff+expertise&start=60 (Caused by ProxyError('Unable to connect to proxy. Your proxy appears to only use HTTP and not HTTPS, try changing your proxy URL to be HTTP. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#https-proxy-error-http-proxy', SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1006)'))))
2024-06-04 13:56:55,291 - INFO - Removed invalid proxy: 198.199.86.11:8080
2024-06-04 13:56:56,067 - ERROR - Request failed (retries left: 4): HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: /search?q=This+work+was+supported+in+part+through+the+NYU+IT+High+Performance+Computing+resources%2C+services%2C+and+staff+expertise&start=50 (Caused by ProxyError('Unable to connect to proxy', ReadTimeoutError("HTTPSConnectionPool(host='www.google.com', port=443): Read timed out. (read timeout=10)")))
2024-06-04 13:56:56,598 - INFO - Fetched HTML content for page 1
2024-06-04 13:56:56,634 - INFO - Fetched results: [{'Title': 'What is your overall opinion on NYU?', 'Link': 'https://www.quora.com/What-is-your-overall-opinion-on-NYU'}, {'Title': 'Research Services', 'Link': 'https://www.cuit.columbia.edu/research'}, {'Title': 'New York University: NYU - NY: Silver School of Social Work', 'Link': 'https://apply.interfolio.com/138951'}, {'Title': 'Job Seeker FAQ', 'Link': 'https://jobs.nyulangone.org/job-seeker-faqs/'}, {'Title': 'Research Data Services in US Higher Education', 'Link': 'https://sr.ithaka.org/publications/research-data-services-in-us-higher-education/'}, {'Title': 'Fashion Institute of Technology', 'Link': 'https://www.fitnyc.edu/'}, {'Title': 'Genome Technology Center', 'Link': 'https://med.nyu.edu/research/scientific-cores-shared-resources/genome-technology-center'}, {'Title': 'HPC Computational Scientist in the The Center ...', 'Link': 'https://apply.interfolio.com/141214'}, {'Title': 'The New School: A University in New York City', 'Link': 'https://www.newschool.edu/'}, {'Title': 'HPC solution | Google Cloud', 'Link': 'https://cloud.google.com/solutions/hpc'}]
2024-06-04 13:56:57,469 - INFO - Fetched HTML content for page 0 without proxy
2024-06-04 13:56:57,496 - INFO - Fetched results: [{'Title': 'Acknowledgement Statement - hpc@nyu.edu', 'Link': 'https://sites.google.com/nyu.edu/nyu-hpc/about/acknowledgement-statement'}, {'Title': 'ABC: A Big CAD Model Dataset For Geometric Deep ...', 'Link': 'https://archive.nyu.edu/handle/2451/43778'}, {'Title': 'ABC Dataset Chunk 0010', 'Link': 'https://archive.nyu.edu/handle/2451/44326'}, {'Title': 'High Performance Computing', 'Link': 'https://www.nyu.edu/life/information-technology/research-computing-services/high-performance-computing.html'}, {'Title': 'High Performance Computing Core', 'Link': 'https://med.nyu.edu/research/scientific-cores-shared-resources/high-performance-computing-core'}, {'Title': 'What is your overall opinion on NYU?', 'Link': 'https://www.quora.com/What-is-your-overall-opinion-on-NYU'}, {'Title': 'I just got access to my universities HPC cluster as an ...', 'Link': 'https://www.reddit.com/r/HPC/comments/ppfwo5/i_just_got_access_to_my_universities_hpc_cluster/'}, {'Title': 'Which is better, NYU or Columbia University?', 'Link': 'https://www.quora.com/Which-is-better-NYU-or-Columbia-University'}, {'Title': 'The New School: A University in New York City', 'Link': 'https://www.newschool.edu/'}, {'Title': 'Fashion Institute of Technology', 'Link': 'https://www.fitnyc.edu/'}]
2024-06-04 13:56:57,672 - INFO - Fetched HTML content for page 7
2024-06-04 13:56:57,699 - WARNING - No search results found on the page.
2024-06-04 13:56:57,701 - INFO - Fetched results: []
2024-06-04 13:56:58,478 - INFO - Fetched HTML content for page 8
2024-06-04 13:56:58,489 - WARNING - No search results found on the page.
2024-06-04 13:56:58,490 - INFO - Fetched results: []
2024-06-04 13:56:58,753 - INFO - Fetched HTML content for page 9
2024-06-04 13:56:58,761 - WARNING - No search results found on the page.
2024-06-04 13:56:58,763 - INFO - Fetched results: []
2024-06-04 13:57:02,346 - INFO - Fetched HTML content for page 2 without proxy
2024-06-04 13:57:02,370 - INFO - Fetched results: [{'Title': 'Career Opportunities', 'Link': 'https://jobs.nyulangone.org/career-opportunities/'}, {'Title': 'Research, Ethics, Compliance, and Safety Training', 'Link': 'https://about.citiprogram.org/'}, {'Title': 'Data Services - Research Guides at New York University', 'Link': 'https://guides.nyu.edu/dataservices/guidelines'}, {'Title': "Alvinya Bohora's Post", 'Link': 'https://www.linkedin.com/posts/alvinyabohora_alvinya-nyu-nyusps-activity-7067424958972358656-XXJ4'}, {'Title': 'Parsons School of Design', 'Link': 'https://www.newschool.edu/parsons/'}, {'Title': 'NYC Health + Hospitals - public health care system in the US', 'Link': 'https://www.nychealthandhospitals.org/'}, {'Title': 'New Mexico Department of Workforce Solutions > Home', 'Link': 'https://www.dws.state.nm.us/en-us/'}, {'Title': 'Human Capital Definition - Economy', 'Link': 'https://www.investopedia.com/terms/h/humancapital.asp'}, {'Title': 'Digital and Data Services: High Performance Computing', 'Link': 'https://library.lclark.edu/digital/hpc'}]
2024-06-04 13:57:02,440 - INFO - Fetched HTML content for page 5
2024-06-04 13:57:02,458 - WARNING - No search results found on the page.
2024-06-04 13:57:02,458 - INFO - Fetched results: []
2024-06-04 13:57:07,322 - INFO - Fetched HTML content for page 6
2024-06-04 13:57:07,343 - INFO - Fetched results: [{'Title': 'MS in Biomedical Informatics', 'Link': 'https://med.nyu.edu/research/vilcek-institute-graduate-biomedical-sciences/ms-biomedical-informatics'}, {'Title': 'Biomedical Imaging & Technology PhD Training Program', 'Link': 'https://med.nyu.edu/research/vilcek-institute-graduate-biomedical-sciences/phd-program/phd-training-programs/biomedical-imaging-technology'}, {'Title': 'NYU and University of Copenhagen Team Up to Work ...', 'Link': 'https://www.nyu.edu/about/news-publications/news/2024/may/nyu-and-university-of-copenhagen-team-up-to-work-toward-supercon.html'}, {'Title': 'NYU Calendar', 'Link': 'https://events.nyu.edu/'}, {'Title': 'Neuroscience Research Associates Program', 'Link': 'https://med.nyu.edu/departments-institutes/neuroscience/education/research-associates-program'}, {'Title': 'Computer Science, M.S.', 'Link': 'https://engineering.nyu.edu/academics/programs/computer-science-ms'}, {'Title': 'Academic Policies - Undergraduate', 'Link': 'https://www.stern.nyu.edu/portal-partners/current-students/undergraduate/academics/academic-policies'}, {'Title': 'ESG and Financial Performance', 'Link': 'https://www.stern.nyu.edu/experience-stern/about/departments-centers-initiatives/centers-of-research/center-sustainable-business/research/research-initiatives/esg-and-financial-performance'}, {'Title': 'Benchmarking of Machine Learning Ocean Subgrid ...', 'Link': 'https://nyuscholars.nyu.edu/en/publications/benchmarking-of-machine-learning-ocean-subgrid-parameterizations-'}, {'Title': 'Frequently Asked Questions: Admissions | NYU SPS', 'Link': 'https://www.sps.nyu.edu/homepage/admissions/admissions-faq.html'}]
2024-06-04 13:57:07,344 - INFO - Fetched 49 results.
2024-06-04 13:57:07,346 - INFO - Results written to CSV
